diff --git a/etc/pa_schemer.ml b/etc/pa_schemer.ml
index b9191316..51d38271 100644
--- a/etc/pa_schemer.ml
+++ b/etc/pa_schemer.ml
@@ -1,14 +1,18 @@
-(* camlp5 pa_r.cmo pa_rp.cmo pa_extend.cmo q_MLast.cmo pr_dump.cmo *)
+(* camlp5r *)
 (* pa_scheme.ml,v *)
 (* Copyright (c) INRIA 2007-2017 *)
 
+#load "pa_rp.cmo";
+#load "pa_extend.cmo";
+#load "q_MLast.cmo";
+
 open Pcaml;
 open Exparser;
 open Versdep;
 
-type choice α β =
-  [ Left of α
-  | Right of β ]
+type choice 'a 'b =
+  [ Left of 'a
+  | Right of 'b ]
 ;
 
 (* Buffer *)
