diff --git a/etc/pr_depend.ml b/etc/pr_depend.ml
index e7cbe65f..f21fa110 100644
--- a/etc/pr_depend.ml
+++ b/etc/pr_depend.ml
@@ -110,7 +110,7 @@ and patt_module =
 and label_patt (p1, p2) = do { patt p1; patt p2 }
 and expr =
   fun
-  [ <:expr< $lid:s$.$e2$ >> -> do { expr_module e2 }
+  [ <:expr< $lid:s$.$e2$ >> -> expr_module e2
   | <:expr< $e1$.$e2$ >> -> do { expr_module e1; expr e2 }
   | ExApp _ e1 e2 -> do { expr e1; expr e2 }
   | ExAre _ e1 e2 -> do { expr e1; expr e2 }
@@ -179,9 +179,8 @@ and module_type =
       module_type mt1;
       module_type mt2
     }
-  | <:module_type< functor () -> $mt2$ >> -> do {
+  | <:module_type< functor () -> $mt2$ >> ->
       module_type mt2
-    }
   | <:module_type< sig $list:sil$ end >> -> list sig_item sil
   | <:module_type< $uid:_$ >> -> ()
   | <:module_type< $mt$ with $list:wc$ >> -> do {
@@ -277,14 +276,12 @@ value find_in_path path name =
   if not (Filename.is_implicit name) then
     if Sys.file_exists name then name else raise Not_found
   else
-    let rec try_dir =
+    try_dir path where rec try_dir =
       fun
       [ [] -> raise Not_found
       | [dir :: rem] ->
           let fullname = Filename.concat dir name in
           if Sys.file_exists fullname then fullname else try_dir rem ]
-    in
-    try_dir path
 ;
 
 value find_depend modname (byt_deps, opt_deps) =
@@ -321,7 +318,7 @@ value print_depend target_file deps =
   | _ -> do {
       print_string target_file;
       print_string depends_on;
-      let rec print_items pos =
+      print_items (String.length target_file + 2) deps where rec print_items pos =
         fun
         [ [] -> print_string "\n"
         | [dep :: rem] ->
@@ -336,8 +333,6 @@ value print_depend target_file deps =
               if rem <> [] then print_string " " else ();
               print_items (String.length dep + 5) rem
             } ]
-      in
-      print_items (String.length target_file + 2) deps
     } ]
 ;
 
