diff --git a/etc/pa_macro_print.ml b/etc/pa_macro_print.ml
index 93c3f6a5..506f0c9e 100644
--- a/etc/pa_macro_print.ml
+++ b/etc/pa_macro_print.ml
@@ -34,9 +34,9 @@ EXTEND
     ]
   ;
   else_print_rule_or_ifdef:
-    [ [ "ELSIFDEF"; e = dexpr; "THEN"; e1 = print_rule_or_ifdef_list ; e2 = else_print_rule_or_ifdef ->
+    [ [ "ELSIFDEF"; e = dexpr; "THEN"; e1 = print_rule_or_ifdef_list ; e2 = SELF ->
           if e then e1 else e2
-      | "ELSIFNDEF"; e = dexpr; "THEN"; e1 = print_rule_or_ifdef_list ; e2 = else_print_rule_or_ifdef ->
+      | "ELSIFNDEF"; e = dexpr; "THEN"; e1 = print_rule_or_ifdef_list ; e2 = SELF ->
           if not e then e1 else e2
       | "ELSE"; e = print_rule_or_ifdef_list -> e ] ]
   ;
