diff --git a/lib/plexing.ml b/lib/plexing.ml
index 947e1980..bf807801 100644
--- a/lib/plexing.ml
+++ b/lib/plexing.ml
@@ -175,7 +175,7 @@ value eval_char s =
 ;
 
 value eval_string loc s =
-  bytes_to_string (loop 0 0) where rec loop len i =
+  let rec loop len i =
     if i = String.length s then get_buff len
     else
       let (len, i) =
@@ -196,6 +196,7 @@ value eval_string loc s =
         else (store len s.[i], i + 1)
       in
       loop len i
+  in bytes_to_string (loop 0 0)
 ;
 
 value default_match =
@@ -219,10 +220,11 @@ value restore_lexing_info = ref None;
 
 value rev_implode l =
   let s = string_create (List.length l) in
-  bytes_to_string (loop (string_length s - 1) l) where rec loop i =
+  let rec loop i =
     fun
     [ [c :: l] -> do { string_unsafe_set s i c; loop (i - 1) l }
     | [] -> s ]
+  in bytes_to_string (loop (string_length s - 1) l)
 ;
 
 module Lexbuf :
