diff --git a/lib/ploc.ml b/lib/ploc.ml
index 30ae4eaa..41cc12eb 100644
--- a/lib/ploc.ml
+++ b/lib/ploc.ml
@@ -82,7 +82,7 @@ value name = ref "loc";
 
 value from_file fname loc =
   let (bp, ep) = (first_pos loc, last_pos loc) in
-  try
+  try do {
     let ic = open_in_bin fname in
     let strm = Stream.of_channel ic in
     let rec loop fname lin =
@@ -126,12 +126,10 @@ value from_file fname loc =
             check_num (10 * n + Char.code c - Char.code '0') (col + 1) s
         | [: col = spaces col; s :] -> check_quote n col s ]
       in
-      let begin_line =
+      begin_line strm where begin_line =
         parser
         [ [: `'#'; col = spaces 1; s :] -> check_num 0 col s
         | [: a = not_a_line_dir 0 :] -> a ]
-      in
-      begin_line strm
     in
     let r =
       try loop fname 1 with
@@ -139,7 +137,7 @@ value from_file fname loc =
           let bol = bol_pos loc in
           (fname, line_nb loc, bp - bol, ep - bol) ]
     in
-    do { close_in ic; r }
+    close_in ic; r }
   with
   [ Sys_error _ -> (fname, 1, bp, ep) ]
 ;
@@ -153,29 +151,26 @@ value second_line fname ep0 (line, bp) ep = do {
       if bol = bp then (line, ep0)
       else (line, ep - bol)
     }
-    else do {
+    else
       let (line, bol) =
         match input_char ic with
         [ '\n' -> (line + 1, p + 1)
         | _ -> (line, bol) ]
       in
       loop line bol (p + 1)
-    }
 };
 
-value get loc = do {
-  if loc.fname = "" || loc.fname = "-" then do {
+value get loc =
+  if loc.fname = "" || loc.fname = "-" then
     (loc.line_nb, loc.bp - loc.bol_pos, loc.line_nb, loc.ep - loc.bol_pos,
      loc.ep - loc.bp)
-  }
-  else do {
+  else
     let (bl, bc, ec) =
       (loc.line_nb, loc.bp - loc.bol_pos, loc.ep - loc.bol_pos)
     in
     let (el, eep) = second_line loc.fname ec (bl, loc.bp) loc.ep in
     (bl, bc, el, eep, ec - bc)
-  }
-};
+;
 
 value call_with r v f a =
   let saved = r.val in
