diff --git a/lib/pprintf.ml b/lib/pprintf.ml
index 98a04dd3..e99abb1a 100644
--- a/lib/pprintf.ml
+++ b/lib/pprintf.ml
@@ -74,8 +74,7 @@ value sprint_break_all force_newlines pc f fl =
                loop s fl
            | [] -> s ])
     (fun () ->
-       loop (f (if fl = [] then pc else {(pc) with aft = ""})) fl
-       where rec loop s =
+       let rec loop s =
          fun
          [ [(sp, off, f) :: fl] ->
              let s =
@@ -84,5 +83,6 @@ value sprint_break_all force_newlines pc f fl =
                      aft = if fl = [] then pc.aft else ""})
              in
              loop s fl
-         | [] -> s ])
+         | [] -> s ]
+         in loop (f (if fl = [] then pc else {(pc) with aft = ""})) fl)
 ;
