diff --git a/main/prtools.ml b/main/prtools.ml
index 60e35277..b9d97458 100644
--- a/main/prtools.ml
+++ b/main/prtools.ml
@@ -376,10 +376,9 @@ value comment_info s =
     if i >= String.length s then ("", 0, 0)
     else if s.[i] = '\n' then loop (i + 1) (nl_bef + 1) 0
     else if s.[i] = ' ' then loop (i + 1) nl_bef (ind_bef + 1)
-    else do {
+    else
       let s = String.sub s i (String.length s - i) in
       (s, nl_bef, ind_bef)
-    }
 ;
 
 value adjust_comment_indentation ind s nl_bef ind_bef =
@@ -393,7 +392,7 @@ value adjust_comment_indentation ind s nl_bef ind_bef =
     let ind_bef = if nl_bef > 0 then ind_bef else ind in
     let len = i_bef_ind + 1 in
     let olen = Buff.mstore 0 (String.make ind ' ') in
-    bytes_to_string (loop olen 0) where rec loop olen i =
+    let rec loop olen i =
       if i = len then Buff.get olen
       else
         let olen = Buff.store olen s.[i] in
@@ -414,14 +413,16 @@ value adjust_comment_indentation ind s nl_bef ind_bef =
           else (olen, i + 1)
         in
         loop olen i
+in bytes_to_string (loop olen 0)
 ;
 
 value eight_chars = String.make 8 ' ';
 value expand_tabs s =
-  bytes_to_string (loop 0 0) where rec loop len i =
+  let rec loop len i =
     if i = String.length s then Buff.get len
     else if s.[i] = '\t' then loop (Buff.mstore len eight_chars) (i + 1)
     else loop (Buff.store len s.[i]) (i + 1)
+in bytes_to_string (loop 0 0)
 ;
 
 value comm_bef ind loc =
@@ -435,7 +436,7 @@ value no_constructors_arity = ref False;
 value expand_module_prefix m =
   loop where rec loop rev_lel =
     fun
-    [ [(p, e) :: rest] -> do {
+    [ [(p, e) :: rest] ->
         let p =
           match p with
           [ <:patt< $uid:_$.$_$ >> -> p
@@ -444,7 +445,6 @@ value expand_module_prefix m =
               <:patt< $uid:m$.$p$ >> ]
         in
         loop [(p, e) :: rev_lel] rest
-      }
   | [] -> List.rev rev_lel ]
 ;
 
