diff --git a/main/main.ml b/main/main.ml
index a94c3c5a..2219fe7d 100644
--- a/main/main.ml
+++ b/main/main.ml
@@ -188,7 +188,7 @@ value anon_fun x = do {
   file_kind.val := file_kind_of_name x
 };
 
-value parse_options sl = do {
+value parse_options sl =
   let ext_spec_list = Pcaml.arg_spec_list () in
   let arg_spec_list = initial_spec_list @ ext_spec_list in
   match Argl.parse arg_spec_list anon_fun sl with
@@ -202,7 +202,7 @@ value parse_options sl = do {
       eprintf "Use option -help for usage\n";
       Some 2
     } ]
-};
+;
 
 Pcaml.add_directive "option"
   (fun
