diff --git a/meta/pa_lexer.ml b/meta/pa_lexer.ml
index 561b5e77..e42c8e38 100644
--- a/meta/pa_lexer.ml
+++ b/meta/pa_lexer.ml
@@ -45,11 +45,11 @@ value conv_rules loc rl =
 ;
 
 value mk_lexer loc rl =
-  cparser loc (None, (conv_rules loc rl))
+  cparser loc (None, conv_rules loc rl)
 ;
 
 value mk_lexer_match loc e rl =
-  cparser_match loc e (None, (conv_rules loc rl))
+  cparser_match loc e (None, conv_rules loc rl)
 ;
 
 (* group together consecutive rules just containing one character *)
