diff --git a/ocpp/ocpp.ml b/ocpp/ocpp.ml
index 19b0e81a..0eb05e35 100644
--- a/ocpp/ocpp.ml
+++ b/ocpp/ocpp.ml
@@ -91,7 +91,7 @@ and inside_string cs =
   | [: :] -> raise (Stream.Error "end of file in string") ]
 ;
 
-value copy_quot cs = do { copy cs; flush stdout; };
+value copy_quot cs = do { copy cs; flush stdout };
 
 value loc_fmt =
   match Sys.os_type with
@@ -100,9 +100,9 @@ value loc_fmt =
   | _ -> ("File \"%s\", line %d, characters %d-%d:\n" : format 'a 'b 'c) ]
 ;
 
-value print_location loc file =
+value print_location loc file = do {
   let (file, line, c1, c2) = Ploc.from_file file loc in
-  do { Printf.eprintf loc_fmt file line c1 c2; flush stderr; }
+  Printf.eprintf loc_fmt file line c1 c2; flush stderr }
 ;
 
 value file = ref "";
