--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/asmcomp/amd64/scheduling.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/asmcomp!amd64!scheduling.ml.INPUT	2020-02-15 20:18:36.533807516 -0800
@@ -10,7 +10,7 @@
 (*                                                                     *)
 (***********************************************************************)
 
-let _ = let module M = Schedgen in () (* to create a dependency *)
+let module M = Schedgen in () ;; (* to create a dependency *)
 
 (* Scheduling is turned off because the processor schedules dynamically
    much better than what we could do. *)
