--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/asmcomp/selectgen.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/asmcomp!selectgen.ml.INPUT	2020-02-15 20:42:44.583807516 -0800
@@ -833,5 +833,4 @@
   let (loc_arg, stack_ofs) = Proc.loc_arguments args in
   stack_ofs = 0
 
-let _ =
-  Simplif.is_tail_native_heuristic := is_tail_call
+  Simplif.is_tail_native_heuristic := is_tail_call ;;
