--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/asmcomp/emitaux.ml	2013-06-03 11:03:59.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/asmcomp!emitaux.ml.INPUT	2020-02-15 20:40:55.940807516 -0800
@@ -205,8 +205,7 @@
 (* Emit debug information *)
 
 (* This assoc list is expected to be very short *)
-let file_pos_nums =
-  (ref [] : (string * int) list ref)
+let file_pos_nums : (string * int) list ref = ref []
 
 (* Number of files *)
 let file_pos_num_cnt = ref 1
