--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/asmcomp/cmmgen.ml	2013-05-22 06:59:24.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/asmcomp!cmmgen.ml.INPUT	2020-02-15 20:37:48.787807516 -0800
@@ -1023,7 +1023,7 @@
 
 (* Translate an expression *)
 
-let functions = (Queue.create() : ufunction Queue.t)
+let functions : ufunction Queue.t = Queue.create()
 
 let rec transl = function
     Uvar id ->
