--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/asmcomp/asmlink.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/asmcomp!asmlink.ml.INPUT	2020-02-15 20:35:13.824807516 -0800
@@ -120,7 +120,7 @@
 
 (* First pass: determine which units are needed *)
 
-let missing_globals = (Hashtbl.create 17 : (string, string list ref) Hashtbl.t)
+let missing_globals : (string, string list ref) Hashtbl.t = Hashtbl.create 17
 
 let is_required name =
   try ignore (Hashtbl.find missing_globals name); true
