--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/bytecomp/translcore.ml	2013-05-28 04:05:58.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/bytecomp!translcore.ml.INPUT	2020-02-15 20:51:41.445807516 -0800
@@ -636,7 +636,7 @@
   | Texp_function (_, pat_expr_list, partial) ->
       let ((kind, params), body) =
         event_function e
-          (function repr ->
+          (fun repr ->
             let pl = push_defaults e.exp_loc [] pat_expr_list partial in
             transl_function e.exp_loc !Clflags.native_code repr partial pl)
       in
