--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/bytecomp/bytegen.ml	2012-11-29 01:55:00.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/bytecomp!bytegen.ml.INPUT	2020-02-15 20:43:37.114807516 -0800
@@ -265,7 +265,7 @@
     rec_vars: Ident.t list;             (* mutually recursive fn names *)
     rec_pos: int }                      (* rank in recursive definition *)
 
-let functions_to_compile  = (Stack.create () : function_to_compile Stack.t)
+let functions_to_compile  : function_to_compile Stack.t = Stack.create ()
 
 (* Name of current compilation unit (for debugging events) *)
 
