--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/bytecomp/simplif.ml	2013-03-09 14:38:52.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/bytecomp!simplif.ml.INPUT	2020-02-15 20:50:30.884807516 -0800
@@ -292,7 +292,7 @@
 
   (* First pass: count the occurrences of all let-bound identifiers *)
 
-  let occ = (Hashtbl.create 83: (Ident.t, int ref) Hashtbl.t) in
+  let occ: (Ident.t, int ref) Hashtbl.t = Hashtbl.create 83 in
   (* The global table [occ] associates to each let-bound identifier
      the number of its uses (as a reference):
      - 0 if never used
