--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/bytecomp/bytepackager.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/bytecomp!bytepackager.ml.INPUT	2020-02-15 20:45:59.626807516 -0800
@@ -103,7 +103,7 @@
         raise(Error(Not_an_object_file file));
       let compunit_pos = input_binary_int ic in
       seek_in ic compunit_pos;
-      let compunit = (input_value ic : compilation_unit) in
+      let compunit : compilation_unit = input_value ic in
       if compunit.cu_name <> name
       then raise(Error(Illegal_renaming(name, file, compunit.cu_name)));
       close_in ic;
