--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/input_handling.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!input_handling.ml.INPUT	2020-02-15 21:02:34.232807516 -0800
@@ -71,7 +71,7 @@
             select (List.map fst !active_files) [] [] (-1.)
           in
             List.iter
-              (function fd ->
+              (fun fd ->
                  let (funct, iochan) = (List.assoc fd !active_files) in
                    funct iochan)
               input
