--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/trap_barrier.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!trap_barrier.ml.INPUT	2020-02-15 21:13:18.941807516 -0800
@@ -28,7 +28,7 @@
 let update_trap_barrier () =
   if !current_checkpoint.c_trap_barrier <> !current_trap_barrier then
     Exec.protect
-      (function () ->
+      (fun () ->
          set_trap_barrier !current_trap_barrier;
          !current_checkpoint.c_trap_barrier <- !current_trap_barrier)
 
