--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/show_information.ml	2012-10-17 05:26:42.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!show_information.ml.INPUT	2020-02-15 21:09:01.621807516 -0800
@@ -47,7 +47,7 @@
              fprintf ppf "Breakpoints: %a@."
              (fun ppf l ->
                List.iter
-                (function x -> fprintf ppf "%i " x) l)
+                (fun x -> fprintf ppf "%i " x) l)
              (List.sort compare breakpoints));
         show_point ev true
   | Some {rep_type = Exited} ->
@@ -92,7 +92,7 @@
       | breakpoints ->
           fprintf ppf "Breakpoints: %a@."
           (fun ppf l ->
-            List.iter (function x -> fprintf ppf "%i " x) l)
+            List.iter (fun x -> fprintf ppf "%i " x) l)
           (List.sort compare breakpoints);
       end;
       show_point sel_ev selected
