--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/debugcom.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!debugcom.ml.INPUT	2020-02-15 20:58:18.660807516 -0800
@@ -83,7 +83,7 @@
   Input_handling.execute_with_other_controller
     Input_handling.exit_main_loop
     !conn
-    (function () ->
+    (fun () ->
        Input_handling.main_loop ();
        let summary =
          match input_char !conn.io_in with
