--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/main.ml	2013-03-09 14:38:52.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/debugger!main.ml.INPUT	2020-02-15 21:03:29.014807516 -0800
@@ -38,7 +38,7 @@
     loop ppf
   with
   | End_of_file ->
-      protect ppf restart (function ppf ->
+      protect ppf restart (fun ppf ->
         forget_process
           !current_checkpoint.c_fd
           !current_checkpoint.c_pid;
@@ -46,14 +46,14 @@
         stop_user_input ();
         restart ppf)
   | Toplevel ->
-      protect ppf restart (function ppf ->
+      protect ppf restart (fun ppf ->
         pp_print_flush ppf ();
         stop_user_input ();
         restart ppf)
   | Sys.Break ->
-      protect ppf restart (function ppf ->
+      protect ppf restart (fun ppf ->
         fprintf ppf "Interrupted.@.";
-        Exec.protect (function () ->
+        Exec.protect (fun () ->
           stop_user_input ();
           if !loaded then begin
             try_select_frame 0;
@@ -61,7 +61,7 @@
           end);
         restart ppf)
   | Current_checkpoint_lost ->
-      protect ppf restart (function ppf ->
+      protect ppf restart (fun ppf ->
         fprintf ppf "Trying to recover...@.";
         stop_user_input ();
         recover ();
@@ -69,7 +69,7 @@
         show_current_event ppf;
         restart ppf)
   | Current_checkpoint_lost_start_at (time, init_duration) ->
-      protect ppf restart (function ppf ->
+      protect ppf restart (fun ppf ->
         let b =
           if !current_duration = -1L then begin
             let msg = sprintf "Restart from time %Ld and try to get \
@@ -228,6 +228,6 @@
       Cmi_format.report_error err_formatter e;
       eprintf "@]@.";
       exit 2
+;;
 
-let _ =
-  Printexc.catch (Unix.handle_unix_error main) ()
+  Printexc.catch (Unix.handle_unix_error main) () ;;
