--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/source.ml	2013-03-09 14:38:52.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/debugger!source.ml.INPUT	2020-02-15 21:09:36.978807516 -0800
@@ -80,8 +80,7 @@
         (list_truncate !buffer_max_count ((mdle, buffer)::!buffer_list));
       buffer
 
-let buffer_content =
-  (fst : buffer -> string)
+let buffer_content : buffer -> string = fst
 
 let buffer_length x =
   String.length (buffer_content x)
