--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/printval.ml	2012-10-17 05:26:42.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!printval.ml.INPUT	2020-02-15 21:06:13.667807516 -0800
@@ -20,8 +20,7 @@
 
 (* To name printed and ellipsed values *)
 
-let named_values =
-  (Hashtbl.create 29 : (int, Debugcom.Remote_value.t * type_expr) Hashtbl.t)
+let named_values : (int, Debugcom.Remote_value.t * type_expr) Hashtbl.t = Hashtbl.create 29
 let next_name = ref 1
 
 let reset_named_values () =
