--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/breakpoints.ml	2013-03-09 14:38:52.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/debugger!breakpoints.ml.INPUT	2020-02-15 20:54:27.796807516 -0800
@@ -65,7 +65,7 @@
    []
   end
     @
-  List.map fst (List.filter (function (_, {ev_pos = pos}) -> pos = pc)
+  List.map fst (List.filter (fun (_, {ev_pos = pos}) -> pos = pc)
                             !breakpoints)
 
 (* Is there a breakpoint at `pc' ? *)
@@ -79,7 +79,7 @@
   if !debug_breakpoints then
     (print_string "Removing breakpoints..."; print_newline ());
   List.iter
-    (function (pos, _) ->
+    (fun (pos, _) ->
        if !debug_breakpoints then begin
          print_int pos;
          print_newline()
@@ -93,7 +93,7 @@
   if !debug_breakpoints then
     (print_string "Setting breakpoints..."; print_newline ());
   List.iter
-    (function (pos, _) ->
+    (fun (pos, _) ->
        if !debug_breakpoints then begin
          print_int pos;
          print_newline()
@@ -112,14 +112,14 @@
   end;
   if !current_checkpoint.c_breakpoint_version <> !current_version then
     Exec.protect
-      (function () ->
+      (fun () ->
          remove_breakpoints !current_checkpoint.c_breakpoints;
          set_breakpoints !positions;
          copy_breakpoints ())
 
 let change_version version pos =
   Exec.protect
-    (function () ->
+    (fun () ->
        current_version := version;
        positions := pos)
 
@@ -164,7 +164,7 @@
       new_breakpoint (Symbols.any_event_at_pc !pc)
   | event ->
       Exec.protect
-        (function () ->
+        (fun () ->
            incr breakpoint_number;
            insert_position event.ev_pos;
            breakpoints := (!breakpoint_number, event) :: !breakpoints);
@@ -178,7 +178,7 @@
     let ev = List.assoc number !breakpoints in
     let pos = ev.ev_pos in
       Exec.protect
-        (function () ->
+        (fun () ->
            breakpoints := List.remove_assoc number !breakpoints;
            remove_position pos;
            printf "Removed breakpoint %d at %d: %s" number ev.ev_pos
@@ -191,7 +191,7 @@
       raise Not_found
 
 let remove_all_breakpoints () =
-  List.iter (function (number, _) -> remove_breakpoint number) !breakpoints
+  List.iter (fun (number, _) -> remove_breakpoint number) !breakpoints
 
 (*** Temporary breakpoints. ***)
 
@@ -214,7 +214,7 @@
         end
 
     in
-      Exec.protect (function () -> insert_position pc);
+      Exec.protect (fun () -> insert_position pc);
       temporary_breakpoint_position := Some pc;
       try
         funct ();
