--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/debugger/pattern_matching.ml	2013-03-22 11:18:26.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/debugger!pattern_matching.ml.INPUT	2020-02-15 21:04:21.011807516 -0800
@@ -107,7 +107,7 @@
              in
                if value_tag last <> 0 then error_matching ();
                flat_map
-                 (function (x, y) -> pattern_matching x y ty_arg)
+                 (fun (x, y) -> pattern_matching x y ty_arg)
                  (rev list)
          | P_nth (n, patt) ->
              let rec find k current =
@@ -209,7 +209,7 @@
         let lbl =
           try
             primitives__find
-              (function l -> same_name l label)
+              (fun l -> same_name l label)
               label_list
           with Not_found ->
               prerr_endline "Label not found.";
@@ -241,7 +241,7 @@
       Invalid_argument _ -> error_matching ()
   in
     flat_map
-      (function (x, y, z) -> pattern_matching x y z)
+      (fun (x, y, z) -> pattern_matching x y z)
       (rev
          (snd
             (it_list
