--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/lex/compact.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/lex!compact.ml.INPUT	2020-02-15 21:14:46.917807516 -0800
@@ -45,9 +45,9 @@
       let r = do_emit_code c in
       Hashtbl.add memory c r ;
       r
-
+;;
 (* Code address 0 is the empty code (ie do nothing) *)
-let _ = mem_emit_code []
+mem_emit_code [] ;;
 
 let emit_tag_code c = mem_emit_code (List.fold_right ins_tag c [])
 and emit_mem_code c  =mem_emit_code (List.fold_right ins_mem c [])
