--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_print.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_print.ml.INPUT	2020-02-15 21:32:49.198807516 -0800
@@ -23,8 +23,8 @@
   in
   (fmt, flush)
 
-let (type_fmt, flush_type_fmt) = new_fmt ()
-let _ =
+let (type_fmt, flush_type_fmt) = new_fmt () ;;
+
   let (out, flush, outnewline, outspace) =
     pp_get_all_formatter_output_functions type_fmt ()
   in
@@ -32,6 +32,7 @@
     ~out ~flush
     ~newline: (fun () -> out "\n  " 0 3)
     ~spaces: outspace
+;;
 
 let (modtype_fmt, flush_modtype_fmt) = new_fmt ()
 
