--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/shell.ml	2012-10-17 14:03:19.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!shell.ml.INPUT	2020-02-15 21:25:06.268807516 -0800
@@ -42,8 +42,6 @@
       let st = sys_command cmd in
       if st <> 0 then
         failwith (Printf.sprintf "Error during command `%s'.\nExit code %d.\n" cmd st)
-      else
-        ()
     end
   else
     match My_unix.execute_many ~ticker:Log.update ~display:Log.display [[(fun () -> cmd)]] with
