--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/glob.ml	2013-05-28 05:16:10.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!glob.ml.INPUT	2020-02-15 21:18:51.419807516 -0800
@@ -38,7 +38,7 @@
 (*** match_character_class *)
 let match_character_class cl c =
   Bool.eval
-    begin function (c1,c2) ->
+    begin fun (c1,c2) ->
       c1 <= c && c <= c2
     end
     cl
