--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/generators/odoc_todo.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!generators!odoc_todo.ml.INPUT	2020-02-15 21:28:03.913807516 -0800
@@ -218,7 +218,8 @@
        scanner <- new scanner self
   end
 end
+;;
 
-let _ = Odoc_args.set_generator
+Odoc_args.set_generator
  (Odoc_gen.Html (module Generator : Odoc_html.Html_generator))
  ;;
