--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc.ml	2013-05-28 04:04:11.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc.ml.INPUT	2020-02-15 21:28:45.457807516 -0800
@@ -39,8 +39,9 @@
         iter (files, incs) q
   in
   iter ([], []) arg_list
+;;
 
-let _ = print_DEBUG "Fin analyse des arguments pour le dynamic load"
+print_DEBUG "Fin analyse des arguments pour le dynamic load" ;;
 
 (** Return the real name of the file to load,
    searching it in the paths if it is
@@ -105,8 +106,7 @@
     )
 
 let modules = Odoc_analyse.analyse_files ~init: loaded_modules !Odoc_global.files
-
-let _ =
+;;
   match !Odoc_global.dump with
     None -> ()
   | Some f ->
@@ -114,9 +114,8 @@
       with Failure s ->
         prerr_endline s ;
         incr Odoc_global.errors
+;;
 
-
-let _ =
   match !Odoc_args.current_generator with
     None ->
       ()
@@ -125,8 +124,8 @@
       Odoc_info.verbose Odoc_messages.generating_doc;
       generator#generate modules;
       Odoc_info.verbose Odoc_messages.ok
+;;
 
-let _ =
   if !Odoc_global.errors > 0 then
   (
    prerr_endline (Odoc_messages.errors_occured !Odoc_global.errors) ;
@@ -134,3 +133,4 @@
   )
   else
     exit 0
+;;
