--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/ocamlbuild_executor.ml	2012-08-01 07:47:00.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!ocamlbuild_executor.ml.INPUT	2020-02-15 21:22:22.977807516 -0800
@@ -177,8 +177,6 @@
         job.job_dying <- true;
         Queue.add (job, continue) jobs_to_terminate
       end
-    else
-      ()
   in
   (* ***)
   (*** add_more_jobs_if_possible *)
@@ -213,8 +211,6 @@
                 Buffer.add_substring job.job_buffer u 0 m;
                 if loop then
                   iteration ()
-                else
-                  ()
               end
           in
           iteration ()
