--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/plugin.ml	2013-09-06 04:36:18.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!plugin.ml.INPUT	2020-02-15 21:24:25.047807516 -0800
@@ -248,8 +248,6 @@
           let rc = sys_command (Command.string_of_command_spec spec) in
           raise (Exit_silently_with_code rc);
         end
-      else
-        ()
   end
 ;;
 
