--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_dep.ml	2013-03-19 00:22:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_dep.ml.INPUT	2020-02-15 21:29:23.184807516 -0800
@@ -116,14 +116,10 @@
              node.near <- S.remove child node.near ;
              node.far <- List.filter (fun (ch,_) -> ch <> child) node.far
             )
-          else
-            ()
         )
         node.near;
       if node.reflex then
         node.near <- S.add node.id node.near
-      else
-        ()
 
     let kernel graph =
       (* compute transitive closure *)
