--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_info.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_info.ml.INPUT	2020-02-15 21:31:43.404807516 -0800
@@ -174,8 +174,6 @@
 let verbose s =
   if !Odoc_global.verbose then
     (print_string s ; print_newline ())
-  else
-    ()
 
 let warning s = Odoc_global.pwarning s
 let print_warnings = Odoc_config.print_warnings
