--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/examples/example3/epoch.ml	2012-08-01 07:47:00.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!examples!example3!epoch.ml.INPUT	2020-02-15 21:18:25.830807516 -0800
@@ -10,7 +10,7 @@
 (*                                                                     *)
 (***********************************************************************)
 
-let _ =
+
   let s = Num.num_of_string (Printf.sprintf "%.0f" (Unix.gettimeofday ())) in
   let ps = Num.mult_num (Num.num_of_string "1000000000000") s in
   Printf.printf "%s picoseconds have passed since January 1st, 1970.\n"
