--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_latex.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_latex.ml.INPUT	2020-02-15 21:32:10.372807516 -0800
@@ -290,8 +290,6 @@
     method latex_of_Target fmt ~target ~code =
       if String.lowercase target = "latex" then
         self#latex_of_Latex fmt code
-      else
-        ()
 
     method latex_of_Raw fmt s =
       ps fmt (self#escape s)
