--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_man.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_man.ml.INPUT	2020-02-15 22:02:52.976807516 -0800
@@ -314,7 +314,7 @@
     method man_of_custom_text b s t = ()
 
     method man_of_Target b ~target ~code =
-      if String.lowercase target = "man" then bs b code else ()
+      if String.lowercase target = "man" then bs b code
 
     (** Print groff string to display code. *)
     method man_of_code b s = self#man_of_text b [ Code s ]
