--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/main.ml	2013-08-20 09:02:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!main.ml.INPUT	2020-02-15 21:19:41.104807516 -0800
@@ -229,7 +229,6 @@
       end targets [] in
 
     if !Options.program_to_execute then
-      begin
         match List.rev cmds with
         | [] -> raise (Exit_usage "Using -- requires one target");
         | cmd :: rest ->
@@ -237,9 +236,6 @@
           let cmd_spec = S [P cmd; atomize !Options.program_args] in
           dprintf 3 "Running the user command:@ %a" Pathname.print cmd;
           raise (Exit_with_code (call cmd_spec)) (* Exit with the exit code of the called command *)
-      end
-    else
-      ()
   with
   | Ocaml_dependencies.Circular_dependencies(seen, p) ->
       raise
