--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamlbuild/ocaml_dependencies.ml	2012-08-01 07:47:00.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamlbuild!ocaml_dependencies.ml.INPUT	2020-02-15 21:21:37.823807516 -0800
@@ -202,12 +202,12 @@
         seen := fn :: !seen;
         Resources.iter begin fun f ->
           if sys_file_exists f then
-            if Filename.check_suffix f ".cmi" then
+            if Filename.check_suffix f ".cmi" then (
               let f' = caml_obj_ext_of_cmi f in
               if f' <> fn then
                 if sys_file_exists f' then aux f'
-                else if pack_mode then aux f else ()
-              else ()
+                else if pack_mode then aux f
+            )
             else aux f
         end (dependencies_of fn);
         needed := fn :: !needed
