--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/ocamldoc/odoc_global.ml	2013-06-05 09:34:40.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/ocamldoc!odoc_global.ml.INPUT	2020-02-15 21:29:51.877807516 -0800
@@ -13,7 +13,7 @@
 (** Global variables. *)
 
 (* Tell ocaml compiler not to generate files. *)
-let _ = Clflags.dont_write_files := true
+Clflags.dont_write_files := true ;;
 
 open Clflags
 
