--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/unix/unix.ml	2013-08-01 05:13:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!unix!unix.ml.INPUT	2020-02-15 22:14:14.122807516 -0800
@@ -83,9 +83,9 @@
   | EUNKNOWNERR of int
 
 exception Unix_error of error * string * string
-
-let _ = Callback.register_exception "Unix.Unix_error"
-                                    (Unix_error(E2BIG, "", ""))
+;;
+Callback.register_exception "Unix.Unix_error"
+                                    (Unix_error(E2BIG, "", "")) ;;
 
 external error_message : error -> string = "unix_error_message"
 
@@ -906,7 +906,7 @@
   | Process_out of out_channel
   | Process_full of in_channel * out_channel * in_channel
 
-let popen_processes = (Hashtbl.create 7 : (popen_process, int) Hashtbl.t)
+let popen_processes : (popen_process, int) Hashtbl.t = Hashtbl.create 7
 
 let open_proc cmd proc input output toclose =
   let cloexec = List.for_all try_set_close_on_exec toclose in
