--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/fileopen.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!fileopen.ml.INPUT	2020-02-15 21:43:47.186807516 -0800
@@ -25,7 +25,7 @@
   Button.create cvs
     [Text "Save";
      Command
-       (function _ ->
+       (fun _ ->
          let s =
            getSaveFile
              [Title "SAVE FILE TEST";
@@ -41,7 +41,7 @@
   Button.create cvs
     [Text "Open";
      Command
-       (function _ ->
+       (fun _ ->
           let s = getOpenFile [] in
           Label.configure t [Text s])];;
 
@@ -49,7 +49,7 @@
   Button.create cvs
     [Text "Quit";
      Command
-       (function _ -> closeTk (); exit 0)];;
+       (fun _ -> closeTk (); exit 0)];;
 
 pack [cvs; q;  bb; b; t] [];;
 
