--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/threads/unix.ml	2013-08-01 08:31:25.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!threads!unix.ml.INPUT	2020-02-15 22:13:29.077807516 -0800
@@ -33,12 +33,12 @@
   | Resumed_io
   | Resumed_select of file_descr list * file_descr list * file_descr list
   | Resumed_wait of int * process_status
-
+;;
 (* to avoid warning *)
-let _ = [Resumed_wakeup; Resumed_delay; Resumed_join;
+[Resumed_wakeup; Resumed_delay; Resumed_join;
          Resumed_io; Resumed_select ([], [], []);
          Resumed_wait (0, WEXITED 0)]
-
+;;
 external thread_initialize : unit -> unit = "thread_initialize"
 external thread_wait_read : file_descr -> unit = "thread_wait_read"
 external thread_wait_write : file_descr -> unit = "thread_wait_write"
@@ -54,10 +54,10 @@
 let select_aux arg = thread_select arg
 let wait_pid_aux pid = thread_wait_pid pid
 let delay duration = thread_delay duration
-
+;;
 (* Make sure that threads are initialized (PR#1516). *)
 
-let _ = thread_initialize()
+thread_initialize() ;;
 
 (* Back to the Unix module *)
 
@@ -133,9 +133,9 @@
   | EUNKNOWNERR of int
 
 exception Unix_error of error * string * string
-
-let _ = Callback.register_exception "Unix.Unix_error"
-                                    (Unix_error(E2BIG, "", ""))
+;;
+Callback.register_exception "Unix.Unix_error"
+                                    (Unix_error(E2BIG, "", "")) ;;
 
 external error_message : error -> string = "unix_error_message"
 
@@ -982,7 +982,7 @@
   | Process_out of out_channel
   | Process_full of in_channel * out_channel * in_channel
 
-let popen_processes = (Hashtbl.create 7 : (popen_process, int) Hashtbl.t)
+let popen_processes : (popen_process, int) Hashtbl.t = Hashtbl.create 7
 
 let open_proc cmd proc input output toclose =
   match fork() with
