--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/threads/pervasives.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!threads!pervasives.ml.INPUT	2020-02-15 22:11:17.116807516 -0800
@@ -549,5 +549,5 @@
 
 external register_named_value : string -> 'a -> unit
                               = "caml_register_named_value"
-
-let _ = register_named_value "Pervasives.do_at_exit" do_at_exit
+;;
+register_named_value "Pervasives.do_at_exit" do_at_exit ;;
