--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_labltk/demo.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_labltk!demo.ml.INPUT	2020-02-15 21:51:29.975807516 -0800
@@ -21,9 +21,8 @@
 (* First, open these modules for convenience *)
 open StdLabels
 open Tk
-
+;;
 (* Dummy let *)
-let _ =
 
 (* Initialize Tk *)
 let top = openTk () in
@@ -164,3 +163,4 @@
 
 (* Main Loop *)
 Printexc.print mainLoop ()
+;;
