--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_labltk/calc.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_labltk!calc.ml.INPUT	2020-02-15 21:49:24.709807516 -0800
@@ -125,5 +125,5 @@
 let top = openTk ()
 
 let applet = new calculator ~parent:top
-
-let _ = mainLoop ()
+;;
+mainLoop () ;;
