--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/systhreads/thread.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!systhreads!thread.ml.INPUT	2020-02-15 22:10:18.045807516 -0800
@@ -54,8 +54,8 @@
   match Sys.os_type with
   | "Win32" -> Sys.sigterm
   | _       -> Sys.sigvtalrm
+;;
 
-let _ =
   Sys.set_signal preempt_signal (Sys.Signal_handle preempt);
   thread_initialize();
   at_exit
@@ -67,7 +67,7 @@
            default handler. *)
         Sys.set_signal preempt_signal Sys.Signal_default
     )
-
+;;
 (* Wait functions *)
 
 let delay time = ignore(Unix.select [] [] [] time)
