--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_labltk/hello.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_labltk!hello.ml.INPUT	2020-02-15 21:52:39.787807516 -0800
@@ -28,11 +28,11 @@
 (* Button.create : use of create function defined in button.ml *)
 (* But you shouldn't open Button module for other widget class modules use *)
 let b = Button.create ~text: "Hello, LablTk!" top
-
+;;
 (* Lack of toplevel expressions in lsl, you must use dummy let exp. *)
-let _ = pack [coe b]
+pack [coe b] ;;
 
 (* Last, you must call mainLoop *)
 (* You can write just let _ = mainLoop () *)
 (* But Printexc.print will help you *)
-let _ = Printexc.print mainLoop ()
+Printexc.print mainLoop () ;;
