--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/graph/graphics.ml	2013-07-02 08:14:00.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!graph!graphics.ml.INPUT	2020-02-15 21:36:11.231807516 -0800
@@ -12,11 +12,10 @@
 (***********************************************************************)
 
 exception Graphic_failure of string
-
+;;
 (* Initializations *)
 
-let _ =
-  Callback.register_exception "Graphics.Graphic_failure" (Graphic_failure "")
+  Callback.register_exception "Graphics.Graphic_failure" (Graphic_failure "") ;;
 
 external raw_open_graph: string -> unit = "caml_gr_open_graph"
 external raw_close_graph: unit -> unit = "caml_gr_close_graph"
