--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/frx/frx_synth.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!frx!frx_synth.ml.INPUT	2020-02-15 21:56:52.743807516 -0800
@@ -83,6 +83,6 @@
 (* Remove all callbacks associated to widget <w> *)
 let remove_callbacks w =
   Hashtbl.iter (fun _ h -> Hashtbl.remove h (Widget.name w)) events
+;;
 
-let _ =
-  add_destroy_hook remove_callbacks
+  add_destroy_hook remove_callbacks ;;
