--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/support/slave.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!support!slave.ml.INPUT	2020-02-15 22:05:51.329807516 -0800
@@ -39,9 +39,9 @@
   Dynlink.add_interfaces [] [];
   let s = register_callback Widget.dummy dynload in
   tcl_command (sprintf "proc caml::run {l} {camlcb %s l}" s)
+;;
 
-let _ =
-  Printexc.print init ()
+  Printexc.print init () ;;
 
 (* A typical master program would then
  *   caml::run foo.cmo
