--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/num/nat.ml	2013-09-07 00:06:31.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!num!nat.ml.INPUT	2020-02-15 22:09:13.312807516 -0800
@@ -48,8 +48,8 @@
 external lor_digit_nat: nat -> int -> nat -> int -> unit = "lor_digit_nat"
 external lxor_digit_nat: nat -> int -> nat -> int -> unit = "lxor_digit_nat"
 
-external initialize_nat: unit -> unit = "initialize_nat"
-let _ = initialize_nat()
+external initialize_nat: unit -> unit = "initialize_nat" ;;
+initialize_nat() ;;
 
 let length_nat (n : nat) = Obj.size (Obj.repr n) - 1
 
