--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/browser/editor.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!browser!editor.ml.INPUT	2020-02-15 21:36:47.561807516 -0800
@@ -199,7 +199,7 @@
               block_start := Lexing.lexeme_end buffer :: !block_start
           | END ->
               if !block_start = [] then
-                if !after then pend := pos else ()
+                (if !after then pend := pos)
               else block_start := List.tl !block_start
           | EOF ->
               pend := pos
