--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_labltk/clock.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_labltk!clock.ml.INPUT	2020-02-15 21:49:54.246807516 -0800
@@ -128,6 +128,6 @@
 (* Create a clock on the main window *)
 let clock =
   new clock ~parent:top
-
+;;
 (* Wait for events *)
-let _ = mainLoop ()
+mainLoop () ;;
