--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/tetris.ml	2013-02-27 11:27:19.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!tetris.ml.INPUT	2020-02-15 21:47:12.288807516 -0800
@@ -243,7 +243,7 @@
   done
 ;;
 
-let timer_ref = (ref None : Timer.t option ref);;
+let timer_ref : Timer.t option ref = ref None;;
 
 let remove_timer () =
   match !timer_ref with
@@ -392,7 +392,7 @@
           ignore (sub m)
         end
       end
-    else () in
+ in
 
   let image_load =
     let i =
