--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/mytext.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!mytext.ml.INPUT	2020-02-15 21:45:22.529807516 -0800
@@ -47,7 +47,7 @@
   prerr_endline ("Yanked: " ^ !buffer)
 ;;
 
-let _ =
+
   scroll_link scrollbar text;
 
   pack [text; scrollbar][Side Side_Left; Fill Fill_Y];
