--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/support/rawwidget.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!support!rawwidget.ml.INPUT	2020-02-15 22:05:18.630807516 -0800
@@ -48,11 +48,11 @@
 and text
 and toplevel
 
-let forget_type w = (Obj.magic (w : 'a raw_widget) : raw_any raw_widget)
+let forget_type w : raw_any raw_widget = Obj.magic (w : 'a raw_widget)
 let coe = forget_type
 
 (* table of widgets *)
-let table = (Hashtbl.create 401 : (string, raw_any raw_widget) Hashtbl.t)
+let table : (string, raw_any raw_widget) Hashtbl.t = Hashtbl.create 401
 
 let name = function
     Untyped s -> s
