--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/support/textvariable.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!support!textvariable.ml.INPUT	2020-02-15 22:07:16.208807516 -0800
@@ -113,8 +113,8 @@
     Hashtbl.remove memo w
   with
     Not_found -> ()
-
-let _ = add_destroy_hook freew
+;;
+add_destroy_hook freew ;;
 
 (* Allocate a new variable *)
 let counter = ref 0
