--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/helloworld.ml	2013-02-27 11:20:54.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!helloworld.ml.INPUT	2020-02-15 21:44:18.137807516 -0800
@@ -26,7 +26,7 @@
   Button.create top [
     Text "foobar";
     Command
-      (function () ->
+      (fun () ->
        print_string "foobar";
        print_newline ();
        flush stdout);
