--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/dynlink/extract_crc.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!dynlink!extract_crc.ml.INPUT	2020-02-15 21:35:43.365807516 -0800
@@ -47,5 +47,6 @@
            "<dir>  Add <dir> to the list of include directories"]
     print_crc usage;
   print_string "\n]\n"
+;;
 
-let _ = main(); exit 0
+main(); exit 0 ;;
