--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/compiler/pp.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!compiler!pp.ml.INPUT	2020-02-15 21:39:34.499807516 -0800
@@ -14,7 +14,6 @@
 (*                                                                     *)
 (***********************************************************************)
 
-let _ =
   try
     let code_list = Ppparse.parse_channel stdin in
     List.iter (Ppexec.exec (fun _ -> ()) print_string) code_list
