--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/compiler/tables.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!compiler!tables.ml.INPUT	2020-02-15 21:42:03.472807516 -0800
@@ -102,17 +102,17 @@
 (******************** The tables ********************)
 
 (* the table of all explicitly defined types *)
-let types_table = (Hashtbl.create 37 : (string, type_def) Hashtbl.t)
+let types_table : (string, type_def) Hashtbl.t = Hashtbl.create 37
 (* "builtin" types *)
 let types_external = ref ([] : (string * parser_arity) list)
 (* dependancy order *)
-let types_order = (Tsort.create () : string Tsort.porder)
+let types_order : string Tsort.porder = Tsort.create ()
 (* Types of atomic values returned by Tk functions *)
 let types_returned = ref ([] : string list)
 (* Function table *)
 let function_table = ref ([] : fullcomponent list)
 (* Widget/Module table *)
-let module_table = (Hashtbl.create 37 : (string, module_def) Hashtbl.t)
+let module_table : (string, module_def) Hashtbl.t = Hashtbl.create 37
 
 
 (* variant name *)
