--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/frx/frx_misc.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!frx!frx_misc.ml.INPUT	2020-02-15 21:56:20.578807516 -0800
@@ -16,7 +16,7 @@
 (* Delayed global, a.k.a cache&carry *)
 let autodef f =
   let v = ref None in
-  (function () ->
+  (fun () ->
      match !v with
        None ->
          let x = f() in
