--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/systhreads/event.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!systhreads!event.ml.INPUT	2020-02-15 22:09:47.089807516 -0800
@@ -72,7 +72,7 @@
   let bev = Array.create (Array.length genev)
                          (fst (genev.(0)) performed condition 0) in
   for i = 1 to Array.length genev - 1 do
-    bev.(i) <- (fst genev.(i)) performed condition i
+    bev.(i) <- fst genev.(i) performed condition i
   done;
   (* See if any of the events is already activable *)
   let rec poll_events i =
