--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/taddition.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!taddition.ml.INPUT	2020-02-15 21:46:24.893807516 -0800
@@ -50,4 +50,4 @@
   Threadtk.mainLoop ()
 ;;
 
-let _ = Printexc.catch main () ;;
+Printexc.catch main () ;;
