--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/otherlibs/labltk/examples_camltk/text.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/otherlibs!labltk!examples_camltk!text.ml.INPUT	2020-02-15 21:48:09.616807516 -0800
@@ -38,14 +38,13 @@
 let yank () =
   Text.insert text (TextIndex (Insert, [])) !buffer []
 
-let _ = bind text [[Control], KeyPressDetail "y"] (BindSet ([], fun _ ->
+bind text [[Control], KeyPressDetail "y"] (BindSet ([], fun _ ->
   yank () ))
 ;;
-let _ = bind text [[Control], KeyPressDetail "k"] (BindSet ([], fun _ ->
+bind text [[Control], KeyPressDetail "k"] (BindSet ([], fun _ ->
   kill () ))
 ;;
 
-let _ =
   scroll_link scrollbar text;
 
   pack [text;f][];
