--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/parsing/pprintast.ml	2013-05-22 05:56:54.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/parsing!pprintast.ml.INPUT	2020-02-15 22:15:55.256807516 -0800
@@ -1098,8 +1098,8 @@
               (fun f l2 -> List.iter (text_x_modtype_x_module f) l2) l2
         | _ -> assert false
   end
-  method type_param f  = function
-    | (a,opt) -> pp f "%s%a" (type_variance a ) self#type_var_option opt
+  method type_param f  = fun
+      (a,opt) -> pp f "%s%a" (type_variance a ) self#type_var_option opt
           (* shared by [Pstr_type,Psig_type]*)
   method  type_def_list f  l =
     let aux f (s, ({ptype_params;ptype_kind;ptype_manifest;ptype_variance;_} as td )) =
