--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/stdlib/pervasives.ml	2013-06-19 04:46:11.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/stdlib!pervasives.ml.INPUT	2020-02-15 22:18:29.331807516 -0800
@@ -456,5 +456,5 @@
 
 external register_named_value : string -> 'a -> unit
                               = "caml_register_named_value"
-
-let _ = register_named_value "Pervasives.do_at_exit" do_at_exit
+;;
+register_named_value "Pervasives.do_at_exit" do_at_exit ;;
