--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/stdlib/parsing.ml	2013-03-09 14:38:52.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/stdlib!parsing.ml.INPUT	2020-02-15 22:18:04.292807516 -0800
@@ -71,10 +71,10 @@
   | Grow_stacks_2
   | Compute_semantic_action
   | Call_error_function
-
+;;
 (* to avoid warnings *)
-let _ = [Read_token; Raise_parse_error; Grow_stacks_1; Grow_stacks_2;
-         Compute_semantic_action; Call_error_function]
+[Read_token; Raise_parse_error; Grow_stacks_1; Grow_stacks_2;
+         Compute_semantic_action; Call_error_function] ;;
 
 external parse_engine :
     parse_tables -> parser_env -> parser_input -> Obj.t -> parser_output
