--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/stdlib/camlinternalOO.ml	2013-03-19 00:22:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/stdlib!camlinternalOO.ml.INPUT	2020-02-15 22:17:31.009807516 -0800
@@ -55,18 +55,18 @@
 
 (**** Items ****)
 
-type item = DummyA | DummyB | DummyC of int
-let _ = [DummyA; DummyB; DummyC 0] (* to avoid warnings *)
+type item = DummyA | DummyB | DummyC of int ;;
+[DummyA; DummyB; DummyC 0] ;; (* to avoid warnings *)
 
-let dummy_item = (magic () : item)
+let dummy_item : item = magic ()
 
 (**** Types ****)
 
 type tag
 type label = int
 type closure = item
-type t = DummyA | DummyB | DummyC of int
-let _ = [DummyA; DummyB; DummyC 0] (* to avoid warnings *)
+type t = DummyA | DummyB | DummyC of int ;;
+[DummyA; DummyB; DummyC 0] ;; (* to avoid warnings *)
 
 type obj = t array
 external ret : (obj -> 'a) -> closure = "%identity"
