--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic-more/testrandom.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic-more!testrandom.ml.INPUT	2020-02-15 22:27:33.038807516 -0800
@@ -11,8 +11,8 @@
 (***********************************************************************)
 
 open Random
+;;
 
-let _ =
   for i = 0 to 20 do
     print_char ' '; print_int (int 1000);
   done;
@@ -20,5 +20,5 @@
   for i = 0 to 20 do
     print_char ' '; print_float (float 1000.);
   done
-
-let _ = exit 0
+;;
+exit 0 ;;
