--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic/boxedints.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic!boxedints.ml.INPUT	2020-02-15 22:30:25.551807516 -0800
@@ -73,8 +73,7 @@
 struct
   open M
   open Ops
-
-  let _ =
+;;
     testing_function "of_int, to_int";
     test 1 (to_int (of_int 0)) 0;
     test 2 (to_int (of_int 123)) 123;
@@ -295,7 +294,7 @@
     test 7 (testcomp max_int min_int)
            (false,true,false,true,false,true,1);
 
-    ()
+    () ;;
 end
 
 (********* Tests on 64-bit arithmetic ***********)
@@ -304,8 +303,7 @@
 struct
   open M
   open Ops
-
-  let _ =
+;;
     testing_function "of_int, to_int";
     test 1 (to_int (of_int 0)) 0;
     test 2 (to_int (of_int 123)) 123;
@@ -513,7 +511,7 @@
     test 7 (testcomp max_int min_int)
            (false,true,false,true,false,true,1);
 
-    ()
+    () ;;
 end
 
 (******** The test proper **********)
@@ -524,8 +522,8 @@
   (a = b, a <> b, a < b, a > b, a <= b, a >= b, compare a b)
 let testcomp_nativeint (a : nativeint) (b : nativeint) =
   (a = b, a <> b, a < b, a > b, a <= b, a >= b, compare a b)
+;;
 
-let _ =
   testing_function "-------- Int32 --------";
   let module A = Test32(struct type t = int32
                                module Ops = Int32
@@ -582,12 +580,13 @@
          (if Sys.word_size = 64
           then Nativeint.of_string "0x123456789ABCDEF0"
           else Nativeint.of_string "0x9ABCDEF0")
-
+;;
 (********* End of test *********)
 
-let _ =
+
   print_newline();
   if !error_occurred then begin
     prerr_endline "************* TEST FAILED ****************"; exit 2
   end else
     exit 0
+;;
