--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-objects/Tests.ml	2012-07-30 11:04:46.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-objects!Tests.ml.INPUT	2020-02-15 23:37:47.741807516 -0800
@@ -270,7 +270,7 @@
 
 let x = ref [];;
 module F(X : sig end) =
-  struct type t = int let _ = (x : < m : t> list ref) end;;
+  struct type t = int ;; (x : < m : t> list ref) ;; end;;
 x;;
 
 type 'a t;;
