--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic/arrays.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic!arrays.ml.INPUT	2020-02-15 22:28:26.367807516 -0800
@@ -133,8 +133,8 @@
   (try
     ignore (Array.sub [|3;4|] max_int 1); print_string "Test 8.3: failed\n"
   with Invalid_argument _ -> ())
+;;
 
-let _ =
   test1();
   test2();
   test3();
@@ -144,3 +144,4 @@
   test7();
   test8();
   exit 0
+;;
