--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/callback/tcallback.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!callback!tcallback.ml.INPUT	2020-02-15 22:36:47.027807516 -0800
@@ -27,7 +27,7 @@
 
 let tak4 x y z u = tak (x, y, z + u)
 
-let raise_exit () = (raise Exit : unit)
+let raise_exit () : unit = raise Exit
 
 let trapexit () =
   begin try
@@ -68,8 +68,8 @@
   (* Allocate some more so that the signal will be tested *)
   let u = (s, s) in
   fst u
+;;
 
-let _ =
   print_int(mycallback1 tak (18, 12, 6)); print_newline();
   print_int(mycallback2 tak2 18 (12, 6)); print_newline();
   print_int(mycallback3 tak3 18 12 6); print_newline();
@@ -79,3 +79,4 @@
   print_string(tripwire mycamlparam); print_newline();
   Sys.set_signal Sys.sigusr1 (Sys.Signal_handle sighandler);
   print_string(callbacksig ()); print_newline()
+;;
