--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc-unsafe/almabench.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc-unsafe!almabench.ml.INPUT	2020-02-15 23:02:06.602807516 -0800
@@ -294,11 +294,11 @@
 
   (* Declination *)
   rdd.(1) <- asin (state.(0).(2) /. rdd.(2)) *. r2d
-
+;;
 
 
 (* Entry point. Calculate RA and Dec for noon on every day in 1900-2100 *)
-let _ =
+
   let jd = [| 0.0; 0.0 |]
   and pv = [| [| 0.0; 0.0; 0.0 |]; [| 0.0; 0.0; 0.0 |] |]
   and position = [| 0.0; 0.0; 0.0 |] in
@@ -322,3 +322,4 @@
       done
     done
   done
+;;
