--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/asmcomp/main.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!asmcomp!main.ml.INPUT	2020-02-15 22:22:39.484807516 -0800
@@ -54,5 +54,5 @@
      "-dscheduling", Arg.Set dump_scheduling, "";
      "-dlinear", Arg.Set dump_linear, ""
     ] compile_file usage
-
-let _ = (*Printexc.catch*) main (); exit 0
+;;
+(*Printexc.catch*) main (); exit 0 ;;
