--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/prim-bswap/bswap.ml	2013-01-06 09:22:09.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!prim-bswap!bswap.ml.INPUT	2020-02-15 23:11:13.620807516 -0800
@@ -23,8 +23,9 @@
            0xf0f0f0f0l]
 let d64 = [0x1122334455667788L;
            0xf0f0f0f0f0f0f0f0L]
+;;
 
-let _ =
   List.iter (fun x -> printf "%x\n" (bswap16 x)) d16;
   List.iter (fun x -> printf "%lx\n" (bswap32 x)) d32;
   List.iter (fun x -> printf "%Lx\n" (bswap64 x)) d64
+;;
