--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc/taku.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc!taku.ml.INPUT	2020-02-15 23:10:46.061807516 -0800
@@ -16,5 +16,5 @@
 
 let rec repeat n =
   if n <= 0 then 0 else tak(18,12,6) + repeat(n-1)
-
-let _ = print_int (repeat 2000); print_newline(); exit 0
+;;
+print_int (repeat 2000); print_newline(); exit 0 ;;
