--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/tool-lexyacc/lexgen.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!tool-lexyacc!lexgen.ml.INPUT	2020-02-15 23:13:29.609807516 -0800
@@ -91,7 +91,7 @@
   Empty
 
 
-let encode_lexdef (Lexdef(_, ld)) =
+let encode_lexdef = function (Lexdef(_, ld)) ->
   chars := [];
   chars_count := 0;
   actions := [];
@@ -194,7 +194,7 @@
     (no_action, [])
 
 
-let memory = (Hashtbl.create 131 : (transition list, int) Hashtbl.t)
+let memory : (transition list, int) Hashtbl.t = Hashtbl.create 131
 let todo = ref ([] : (transition list * int) list)
 let next = ref 0
 
