--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-labels/mixin2.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-labels!mixin2.ml.INPUT	2020-02-15 23:34:04.681807516 -0800
@@ -31,11 +31,11 @@
 
 class ['a] var_ops = object (self : ('a, var) #ops)
   constraint 'a = [> var]
-  method subst ~sub (`Var s as x) =
+  method subst ~sub = function (`Var s as x) ->
     try Subst.find s sub with Not_found -> x
-  method free (`Var s) =
+  method free = function (`Var s) ->
     Names.singleton s
-  method eval (#var as v) = v
+  method eval = function #var as v -> v
 end
 
 (* The lambda language: free variables, substitutions, and evaluation *)
@@ -47,7 +47,7 @@
   fun () -> incr current; !current
 
 class ['a] lambda_ops (ops : ('a,'a) #ops Lazy.t) =
-  let var : 'a var_ops = new var_ops
+  let var = (new var_ops : 'a var_ops)
   and free = lazy !!ops#free
   and subst = lazy !!ops#subst
   and eval = lazy !!ops#eval in
@@ -102,7 +102,7 @@
     | `Neg of 'a | `Mult of 'a * 'a]
 
 class ['a] expr_ops (ops : ('a,'a) #ops Lazy.t) =
-  let var : 'a var_ops = new var_ops
+  let var = (new var_ops : 'a var_ops)
   and free = lazy !!ops#free
   and subst = lazy !!ops#subst
   and eval = lazy !!ops#eval in
@@ -134,7 +134,7 @@
         #var as x -> var#subst ~sub x
       | #expr as e -> self#map ~f:(!!subst ~sub) e
 
-    method eval (#expr as e) =
+    method eval = function (#expr as e) ->
       match self#map ~f:!!eval e with
         `Add(`Num m, `Num n) -> `Num (m+n)
       | `Neg(`Num n) -> `Num (-n)
