--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-gadts/pr5948.ml	2013-03-12 21:59:10.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-gadts!pr5948.ml.INPUT	2020-02-15 23:20:16.137807516 -0800
@@ -26,5 +26,5 @@
     | WrapPoly _ -> intA (* This should not be allowed *)
 ;;
 
-let _ =  example6 (WrapPoly AandBTags) `TagB (* This causes a seg fault *)
+example6 (WrapPoly AandBTags) `TagB (* This causes a seg fault *)
 ;;
