--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/interactive/lib-signals/signals.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!interactive!lib-signals!signals.ml.INPUT	2020-02-15 22:22:01.957807516 -0800
@@ -23,8 +23,8 @@
   print_string "Thank you for pressing ctrl-Z."; print_newline();
   print_string "Now raising an exception..."; print_newline();
   raise Exit
+;;
 
-let _ =
   ignore (Sys.signal Sys.sigint (Sys.Signal_handle break_handler));
   ignore (Sys.signal Sys.sigtstp (Sys.Signal_handle stop_handler));
   begin try
@@ -42,3 +42,4 @@
     print_string "Got Exit, exiting."; print_newline()
   end;
   exit 0
+;;
