--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc-unsafe/quicksort.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc-unsafe!quicksort.ml.INPUT	2020-02-15 23:02:55.064807516 -0800
@@ -86,5 +86,5 @@
 let main () =
   test_sort qsort 500000;
   test_sort qsort2 500000
-
-let _ = main(); exit 0
+;;
+main(); exit 0 ;;
