--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/lib-hashtbl/hfun.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!lib-hashtbl!hfun.ml.INPUT	2020-02-15 22:43:49.149807516 -0800
@@ -16,8 +16,8 @@
      - equal values hash equally, esp NaNs. *)
 
 open Printf
+;;
 
-let _ =
   printf "-- Strings:\n";
   printf "\"\"\t\t%08x\n" (Hashtbl.hash "");
   printf "\"Hello world\"\t%08x\n" (Hashtbl.hash "Hello world");
@@ -51,3 +51,4 @@
   printf "[10..0]\t\t%08x\n" (Hashtbl.hash [10;9;8;7;6;5;4;3;2;1;0]);
 
   ()
+;;
