--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/tool-lexyacc/output.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!tool-lexyacc!output.ml.INPUT	2020-02-15 23:14:40.473807516 -0800
@@ -21,7 +21,7 @@
 
 let copy_buffer = String.create 1024
 
-let copy_chunk (Location(start,stop)) =
+let copy_chunk = function (Location(start,stop)) ->
   seek_in !ic start;
   let tocopy = ref(stop - start) in
   while !tocopy > 0 do
