--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-polyvariants-bugs-2/pr3918c.ml	2012-07-17 08:31:12.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-polyvariants-bugs-2!pr3918c.ml.INPUT	2020-02-15 23:40:22.105807516 -0800
@@ -7,4 +7,4 @@
 open Pr3918b
 
 let f x = (x : 'a vlist :> 'b vlist)
-let f (x : 'a vlist) = (x : 'b vlist)
+let f (x : 'a vlist) : 'b vlist = x
