--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/tool-lexyacc/main.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!tool-lexyacc!main.ml.INPUT	2020-02-15 23:13:57.717807516 -0800
@@ -53,8 +53,8 @@
   output_lexdef header dfa;
   close_in !ic;
   close_out !oc
-
-let _ = main(); exit 0
+;;
+main(); exit 0 ;;
 
 
 (*****
