--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic/patmatch.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic!patmatch.ml.INPUT	2020-02-15 22:33:46.953807516 -0800
@@ -91,8 +91,8 @@
       string_unsafe_set s 3 (unsafe_chr (48 + n mod 10));
       s
     end
+;;
 
-let _ =
   for i = -5 to 10 do printf "f(%d) = %d\n" i (f i) done;
   List.iter (fun i -> printf "g(%d) = %d\n" i (g i))
             [0;300;303;305;400;401;402;999];
@@ -114,7 +114,7 @@
   printf "l([|1|]) = %d\n" (l [|1|]);
   printf "l([|2;3|]) = %d\n" (l [|2;3|]);
   printf "l([|4;5;6|]) = %d\n" (l [|4;5;6|])
-
+;;
 (* PR #5992 *)
 (* Was segfaulting *)
 
