--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc/nucleic.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc!nucleic.ml.INPUT	2020-02-15 23:07:39.471807516 -0800
@@ -264,88 +264,88 @@
   | _ -> false
 
 let
-nuc_C1'
+nuc_C1' = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = c1'
+  -> c1'
 
 let
-nuc_C2
+nuc_C2 = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = c2
+  -> c2
 
 let
-nuc_C3'
+nuc_C3' = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = c3'
+  -> c3'
 
 let
-nuc_C4
+nuc_C4 = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = c4
+  -> c4
 
 let
-nuc_C4'
+nuc_C4' = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = c4'
+  -> c4'
 
 let
-nuc_N1
+nuc_N1 = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = n1
+  -> n1
 
 let
-nuc_O3'
+nuc_O3' = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = o3'
+  -> o3'
 
 let
-nuc_P
+nuc_P = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = p
+  -> p
 
 let
-nuc_dgf_base_tfo
+nuc_dgf_base_tfo = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = dgf_base_tfo
+  -> dgf_base_tfo
 
 let
-nuc_p_o3'_180_tfo
+nuc_p_o3'_180_tfo = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = p_o3'_180_tfo
+  -> p_o3'_180_tfo
 
 let
-nuc_p_o3'_275_tfo
+nuc_p_o3'_275_tfo = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = p_o3'_275_tfo
+  -> p_o3'_275_tfo
 
 let
-nuc_p_o3'_60_tfo
+nuc_p_o3'_60_tfo = function
 (N(dgf_base_tfo,p_o3'_275_tfo,p_o3'_180_tfo,p_o3'_60_tfo,
    p,o1p,o2p,o5',c5',h5',h5'',c4',h4',o4',c1',h1',c2',h2'',o2',h2',
    c3',h3',o3',n1,n3,c2,c4,c5,c6,_))
-  = p_o3'_60_tfo
+  -> p_o3'_60_tfo
 
 let
 rA_N9 = function
@@ -3232,5 +3232,5 @@
 let main () =
   for i = 1 to 50 do ignore(run()) done;
   Printf.printf "%.4f" (run ()); print_newline()
-
-let _ = main ()
+;;
+main () ;;
