--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc/sieve.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc!sieve.ml.INPUT	2020-02-15 23:08:05.618807516 -0800
@@ -46,9 +46,10 @@
 let rec do_list f = function
     [] -> ()
   | a::l -> f a; do_list f l
+;;
 
 
-let _ =
   do_list (fun n -> print_string " "; print_int n) (sieve 50000);
   print_newline();
   exit 0
+;;
