--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic/tailcalls.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic!tailcalls.ml.INPUT	2020-02-15 22:36:05.188807516 -0800
@@ -31,10 +31,11 @@
 
 let indtailcall16 fn a b c d e f g h i j k l m n o p =
   fn a b c d e f g h i j k l m n o p
+;;
 
-let _ =
   print_int (tailcall4 10000000 0 0 0); print_newline();
   print_int (tailcall8 10000000 0 0 0 0 0 0 0); print_newline();
   print_int (tailcall16 10000000 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0); print_newline();
   print_int (indtailcall8 tailcall8 10 0 0 0 0 0 0 0); print_newline();
   print_int (indtailcall16 tailcall16 10 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0); print_newline()
+;;
