--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/misc-kb/kbmain.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!misc-kb!kbmain.ml.INPUT	2020-02-15 23:01:36.060807516 -0800
@@ -74,6 +74,7 @@
 
 let greater pair =
   match group_order pair with Greater -> true | _ -> false
+;;
 
-let _ =
   for i = 1 to 20 do kb_complete greater [] geom_rules done
+;;
