--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-poly-bugs/pr5673_bad.ml	2012-12-26 23:19:12.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-poly-bugs!pr5673_bad.ml.INPUT	2020-02-15 23:38:19.250807516 -0800
@@ -20,4 +20,4 @@
 type refer2 = < poly : 'a 'b 'c . (('b, 'c) #Classdef.cl2 as 'a) >
 
 (* Actually this should succeed ... *)
-let f (x : refer1) = (x : refer2)
+let f (x : refer1) : refer2 = x
