--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic-more/morematch.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic-more!morematch.ml.INPUT	2020-02-15 22:27:04.488807516 -0800
@@ -1090,15 +1090,15 @@
   | C -> 2
 
 let g x = try f x with Match_failure _ -> 3
+;;
 
-let _ =
   test "jacques" g (A `A) 0 ;
   test "jacques" g (A `C) 0 ;
   test "jacques" g (B (`B,`D)) 1 ;
   test "jacaues" g C 2 ;
 (*  test "jacques" g (B (`A,`D)) 3 ; (* type incorrect expected behavior ? *)*)
   ()
-
+;;
 (*
   Compilation bug, segfault, because of incorrect compilation
   of unused match case .. -> "11"
@@ -1121,18 +1121,18 @@
   |  _, _, _, _, _, A, _, _, _, _, B, _, _, _, _, _ -> "11"
   |  B, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _ -> "12"
   |  _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _ -> "13"
-
+;;
 (*
 File "morematch.ml", line 1094, characters 5-51:
 Warning: this match case is unused.
 File "morematch.ml", line 1096, characters 5-51:
 Warning: this match case is unused.
 *)
-let _  =
+
   test "luc"  f (B, A, A, A, A, A, A, A, A, A, A, B, A, A, A, A) "10" ;
   test "luc"  f (B, A, A, A, A, A, A, A, A, A, A, A, A, A, A, A) "12" ;
  ()
-
+;;
 (*
   By Gilles Peskine, compilation raised some assert false i make_failactionneg
 *)
