--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-gadts/pr5985.ml	2013-06-18 06:05:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-gadts!pr5985.ml.INPUT	2020-02-15 19:26:13.794807516 -0800
@@ -9,10 +9,12 @@
 *)
 
 (* Fix it using #-annotations *)
+(* CHET: NO IDEA WTF THIS IS, so just comment it out
 module F (S : sig type #'a s end) = struct
   include S
   type _ t = T : 'a -> 'a s t
-end;; (* syntax error *)
+end;; *)
+(* syntax error *)
 (*
 module M = F (struct type 'a s = int end) ;; (* fail *)
 module M = F (struct type 'a s = new int end) ;; (* ok *)
@@ -52,8 +54,10 @@
 
 (* Another problem with variance *)
 module M = struct type 'a t = 'a -> unit end;;
+(* CHET: another one I got no idea about
 module F(X:sig type #'a t end) =
   struct type +'a s = S of 'b constraint 'a = 'b X.t end;; (* fail *)
+*)
 (*
 module N = F(M);;
 let o = N.S (object end);;
