--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-gadts/pr5906.ml	2013-01-24 20:26:02.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-gadts!pr5906.ml.INPUT	2020-02-15 23:19:44.615807516 -0800
@@ -13,5 +13,5 @@
   | Leq, Int x, Int y -> Bool (x <= y)
   | Leq, Bool x, Bool y -> Bool (x <= y)
   | Add, Int x, Int y -> Int (x + y)
-
-let _ = eval Eq (Int 2) (Int 3)
+;;
+eval Eq (Int 2) (Int 3) ;;
