--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/exotic-syntax/exotic.ml	2013-06-11 00:32:49.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!exotic-syntax!exotic.ml.INPUT	2020-02-15 22:38:51.277807516 -0800
@@ -49,8 +49,8 @@
 function -1n -> () | _ -> ();;
 
 (* surprising places where you can use an operator as a variable name *)
-function (+) -> (+);;
-function _ as (+) -> (+);;
+fun (+) -> (+);;
+fun (_ as (+)) -> (+);;
 for (+) = 0 to 1 do () done;;
 
 (* access a class-type through an extended-module-path *)
@@ -90,7 +90,7 @@
 
 (* let-binding with a type coercion *)
 let x :> int = 1;;
-let x : int :> int = 1;;
+let x = ((1  : int) :> int);;
 
 (* "begin end" as an alias for "()" *)
 begin end;;
