--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/interactive/lib-graph-3/sorts.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!interactive!lib-graph-3!sorts.ml.INPUT	2020-02-15 22:20:19.590807516 -0800
@@ -234,7 +234,7 @@
              "Mergesort", merge_sort, sx, h+2*sy, sx, sy |]
           100 1000;
   close_graph()
+;;
 
-let _ = if !Sys.interactive then () else begin animate(); exit 0 end
-
+if !Sys.interactive then () else begin animate(); exit 0 end
 ;;
