--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/basic-io/wc.ml	2012-10-17 13:09:16.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!basic-io!wc.ml.INPUT	2020-02-15 22:25:33.062807516 -0800
@@ -50,8 +50,8 @@
 let count name =
   count_file name;
   print_result ()
+;;
 
-let _ =
 try
   if Array.length Sys.argv <= 1 then
     count_channel stdin                (* No command-line arguments *)
@@ -64,3 +64,4 @@
   print_string "I/O error: ";
   print_string s;
   print_newline()
+;;
