--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/tool-ocaml/t330-compact-2.ml	2012-07-30 11:04:46.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!tool-ocaml!t330-compact-2.ml.INPUT	2020-02-15 23:15:07.779807516 -0800
@@ -1,6 +1,6 @@
 open Lib;;
 Gc.compact ();;
-let _ = Pervasives.do_at_exit();;
+Pervasives.do_at_exit();;
 
 (**
        0 CONSTINT 42
