--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/testsuite/tests/typing-labels/mixin3.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/testsuite!tests!typing-labels!mixin3.ml.INPUT	2020-02-15 23:34:56.135807516 -0800
@@ -30,11 +30,11 @@
 type var = [`Var of string]
 
 let var = object (self : ([>var], var) #ops)
-  method subst ~sub (`Var s as x) =
+  method subst ~sub = function (`Var s as x) ->
     try Subst.find s sub with Not_found -> x
-  method free (`Var s) =
+  method free = function (`Var s) ->
     Names.singleton s
-  method eval (#var as v) = v
+  method eval = function (#var as v) -> v
 end
 
 (* The lambda language: free variables, substitutions, and evaluation *)
@@ -129,7 +129,7 @@
         #var as x -> var#subst ~sub x
       | #expr as e -> self#map ~f:(!!subst ~sub) e
 
-    method eval (#expr as e) =
+    method eval = function (#expr as e) ->
       match self#map ~f:!!eval e with
         `Add(`Num m, `Num n) -> `Num (m+n)
       | `Neg(`Num n) -> `Num (-n)
