--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/scrapelabels.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/tools!scrapelabels.ml.INPUT	2020-02-15 23:49:22.093807516 -0800
@@ -253,8 +253,8 @@
     close_out oc
   end
   else prerr_endline ("No changes in " ^ name)
+;;
 
-let _ =
   let files = ref [] and intf = ref false
   and keepstd = ref false and keepmore = ref false in
   Arg.parse
@@ -285,3 +285,4 @@
       prerr_endline ("Processing " ^ name);
       Printexc.catch (convert_file ~intf:!intf) name
     end
+;;
