--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/ocamlmktop.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/tools!ocamlmktop.ml.INPUT	2020-02-15 23:46:17.675807516 -0800
@@ -10,8 +10,8 @@
 (*                                                                     *)
 (***********************************************************************)
 
-let _ =
   let args = Ccomp.quote_files (List.tl (Array.to_list Sys.argv)) in
   exit(Sys.command("ocamlc -I +compiler-libs -linkall ocamlcommon.cma \
                     ocamlbytecomp.cma ocamltoplevel.cma "
                    ^ args ^ " topstart.cmo"))
+;;
