--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/objinfo.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/tools!objinfo.ml.INPUT	2020-02-15 23:44:47.213807516 -0800
@@ -219,13 +219,13 @@
   if magic_number = cmo_magic_number then begin
     let cu_pos = input_binary_int ic in
     seek_in ic cu_pos;
-    let cu = (input_value ic : compilation_unit) in
+    let cu : compilation_unit = input_value ic in
     close_in ic;
     print_cmo_infos cu
   end else if magic_number = cma_magic_number then begin
     let toc_pos = input_binary_int ic in
     seek_in ic toc_pos;
-    let toc = (input_value ic : library) in
+    let toc : library = input_value ic in
     close_in ic;
     print_cma_infos toc
   end else if magic_number = cmi_magic_number then begin
@@ -234,12 +234,12 @@
     print_cmi_infos cmi.Cmi_format.cmi_name cmi.Cmi_format.cmi_sign
       cmi.Cmi_format.cmi_crcs
   end else if magic_number = cmx_magic_number then begin
-    let ui = (input_value ic : unit_infos) in
+    let ui : unit_infos = input_value ic in
     let crc = Digest.input ic in
     close_in ic;
     print_cmx_infos (ui, crc)
   end else if magic_number = cmxa_magic_number then begin
-    let li = (input_value ic : library_infos) in
+    let li : library_infos = input_value ic in
     close_in ic;
     print_cmxa_infos li
   end else begin
@@ -274,5 +274,5 @@
 let main() =
   Arg.parse arg_list dump_obj arg_usage;
   exit 0
-
-let _ = main ()
+;;
+main () ;;
