--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/addlabels.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/tools!addlabels.ml.INPUT	2020-02-16 00:15:10.756807516 -0800
@@ -309,7 +309,7 @@
               ~f:(fun m (k,l) -> SMap.add (s.txt^"#"^k) l m)
       in
       ignore (List.fold_left l ~init:values ~f:
-        begin fun values -> function e -> match e.pcf_desc with
+        begin fun values e -> match e.pcf_desc with
           | Pcf_val (s, _, _, e) ->
               add_labels_expr ~text ~classes ~values e;
               SMap.removes [s.txt] values
