--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/read_cmt.ml	2012-08-01 05:38:51.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/tools!read_cmt.ml.INPUT	2020-02-15 23:48:55.221807516 -0800
@@ -59,8 +59,8 @@
       ) (List.sort compare cmt.cmt_imports);
       Printf.printf "%!";
       ()
+;;
 
-let _ =
   Clflags.annotations := true;
 
   Arg.parse arg_list  (fun filename ->
@@ -78,3 +78,4 @@
       Arg.usage arg_list arg_usage
     end
   ) arg_usage
+;;
