--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/tools/tast_iter.ml	2013-01-07 04:31:58.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/tools!tast_iter.ml.INPUT	2020-02-15 23:49:44.388807516 -0800
@@ -90,7 +90,7 @@
     | Texp_newtype _ -> ()
     | Texp_poly cto -> opt (sub # core_type) cto
   in
-  List.iter (function (c, _) -> extra c) exp.exp_extra;
+  List.iter (fun (c, _) -> extra c) exp.exp_extra;
   match exp.exp_desc with
   | Texp_ident _
   | Texp_constant _ -> ()
