--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/toplevel/opttopstart.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/toplevel!opttopstart.ml.INPUT	2020-02-16 00:15:39.516807516 -0800
@@ -10,4 +10,4 @@
 (*                                                                     *)
 (***********************************************************************)
 
-let _ = Opttopmain.main()
+Opttopmain.main() ;;
