--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/toplevel/opttoploop.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/toplevel!opttoploop.ml.INPUT	2020-02-15 23:53:17.573807516 -0800
@@ -212,7 +212,7 @@
 (* The table of toplevel directives.
    Filled by functions from module topdirs. *)
 
-let directive_table = (Hashtbl.create 13 : (string, directive_fun) Hashtbl.t)
+let directive_table : (string, directive_fun) Hashtbl.t = Hashtbl.create 13
 
 (* Execute a toplevel phrase *)
 
@@ -376,18 +376,17 @@
     end else
       len
   end
-
+;;
 (* Toplevel initialization. Performed here instead of at the
    beginning of loop() so that user code linked in with ocamlmktop
    can call directives from Topdirs. *)
 
-let _ =
   Sys.interactive := true;
   Dynlink.init ();
   Compmisc.init_path true;
   Clflags.dlcode := true;
   ()
-
+;;
 let load_ocamlinit ppf =
   match !Clflags.init_file with
   | Some f -> if Sys.file_exists f then ignore (use_silently ppf f)
