--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/typetexp.ml	2013-05-28 04:05:58.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/typing!typetexp.ml.INPUT	2020-02-16 00:07:23.442807516 -0800
@@ -623,7 +623,7 @@
   used_variables := Tbl.empty;
   fun () ->
     List.iter
-      (function (loc, t1, t2) ->
+      (fun (loc, t1, t2) ->
         try unify env t1 t2 with Unify trace ->
           raise (Error(loc, env, Type_mismatch trace)))
       !r
@@ -757,15 +757,15 @@
       fprintf ppf "Unbound row variable in #%a" longident lid
   | Type_mismatch trace ->
       Printtyp.report_unification_error ppf Env.empty trace
-        (function ppf ->
+        (fun ppf ->
            fprintf ppf "This type")
-        (function ppf ->
+        (fun ppf ->
            fprintf ppf "should be an instance of type")
   | Alias_type_mismatch trace ->
       Printtyp.report_unification_error ppf Env.empty trace
-        (function ppf ->
+        (fun ppf ->
            fprintf ppf "This alias is bound to type")
-        (function ppf ->
+        (fun ppf ->
            fprintf ppf "but is used as an instance of type")
   | Present_has_conjunction l ->
       fprintf ppf "The present constructor %s has a conjunctive type" l
