--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/includeclass.ml	2013-02-09 00:42:11.000000000 -0800
+++ patches/ocaml-4.01.0/ALL/typing!includeclass.ml.INPUT	2020-02-16 00:00:14.041807516 -0800
@@ -49,9 +49,9 @@
         "The classes do not have the same number of type parameters"
   | CM_Type_parameter_mismatch (env, trace) ->
       Printtyp.report_unification_error ppf env ~unif:false trace
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "A type parameter has type")
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "but is expected to have type")
   | CM_Class_type_mismatch (env, cty1, cty2) ->
       Printtyp.wrap_printing_env env (fun () ->
@@ -62,21 +62,21 @@
           Printtyp.class_type cty2)
   | CM_Parameter_mismatch (env, trace) ->
       Printtyp.report_unification_error ppf env ~unif:false trace
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "A parameter has type")
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "but is expected to have type")
   | CM_Val_type_mismatch (lab, env, trace) ->
       Printtyp.report_unification_error ppf env ~unif:false trace
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "The instance variable %s@ has type" lab)
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "but is expected to have type")
   | CM_Meth_type_mismatch (lab, env, trace) ->
       Printtyp.report_unification_error ppf env ~unif:false trace
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "The method %s@ has type" lab)
-        (function ppf ->
+        (fun ppf ->
           fprintf ppf "but is expected to have type")
   | CM_Non_mutable_value lab ->
       fprintf ppf
