--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/includemod.ml	2013-06-03 07:46:04.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/typing!includemod.ml.INPUT	2020-02-16 00:00:40.481807516 -0800
@@ -331,8 +331,8 @@
                     (Mtype.strengthen env mty1 path1) mty2)
   with Error reasons ->
     raise Not_found
-
-let _ = Env.check_modtype_inclusion := check_modtype_inclusion
+;;
+Env.check_modtype_inclusion := check_modtype_inclusion ;;
 
 (* Check that an implementation of a compilation unit meets its
    interface. *)
