--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/oprint.ml	2013-07-23 07:48:47.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/typing!oprint.ml.INPUT	2020-02-16 00:01:20.690807516 -0800
@@ -471,10 +471,10 @@
 and print_out_label ppf (name, mut, arg) =
   fprintf ppf "@[<2>%s%s :@ %a@];" (if mut then "mutable " else "") name
     !out_type arg
-
-let _ = out_module_type := print_out_module_type
-let _ = out_signature := print_out_signature
-let _ = out_sig_item := print_out_sig_item
+;;
+out_module_type := print_out_module_type ;;
+out_signature := print_out_signature ;;
+out_sig_item := print_out_sig_item ;;
 
 (* Phrases *)
 
