--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/predef.ml	2013-05-03 06:38:30.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/typing!predef.ml.INPUT	2020-02-16 00:02:26.227807516 -0800
@@ -199,6 +199,6 @@
 (* Start non-predef identifiers at 1000.  This way, more predefs can
    be defined in this file (above!) without breaking .cmi
    compatibility. *)
-
-let _ = Ident.set_current_time 999
+;;
+Ident.set_current_time 999 ;;
 let builtin_idents = List.rev !builtin_idents
