--- /home/chetsky/Hack/Ocaml/GENERIC/4.01.0/.opam-switch/sources/ocaml-base-compiler.4.01.0/typing/subst.ml	2012-10-15 10:50:56.000000000 -0700
+++ patches/ocaml-4.01.0/ALL/typing!subst.ml.INPUT	2020-02-16 00:03:05.789807516 -0800
@@ -199,7 +199,7 @@
 let class_signature s sign =
   { cty_self = typexp s sign.cty_self;
     cty_vars =
-      Vars.map (function (m, v, t) -> (m, v, typexp s t)) sign.cty_vars;
+      Vars.map (fun (m, v, t) -> (m, v, typexp s t)) sign.cty_vars;
     cty_concr = sign.cty_concr;
     cty_inher =
       List.map (fun (p, tl) -> (type_path s p, List.map (typexp s) tl))
