--- /home/chetsky/Hack/Ocaml/GENERIC/4.10.0/.opam-switch/sources/ocaml-base-compiler.4.10.0/testsuite/tests/array-functions/test.ml	2020-02-21 05:50:54.000000000 -0800
+++ patches/ocaml-4.10.0/ALL/testsuite!tests!array-functions!test.ml.INPUT	2020-02-26 18:48:41.990143028 -0800
@@ -105,7 +105,6 @@
   Array.fill f 0 10 1.0;
   assert (Array.for_all (fun a -> a = 1.0) f);
 ;;
-;;
 
 let () =
   let a = [||] in
