--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/asmcomp/schedgen.ml	2012-02-21 09:41:02.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/asmcomp!schedgen.ml.INPUT	2020-02-10 13:40:25.837266054 -0800
@@ -44,11 +44,11 @@
    - code_checkbounds contains the latest checkbound node not matched
      by a subsequent load or store. *)
 
-let code_results = (Hashtbl.create 31 : (location, code_dag_node) Hashtbl.t)
-let code_uses = (Hashtbl.create 31 : (location, code_dag_node) Hashtbl.t)
-let code_stores = ref ([] : code_dag_node list)
-let code_loads = ref ([] : code_dag_node list)
-let code_checkbounds = ref ([] : code_dag_node list)
+let code_results : (location, code_dag_node) Hashtbl.t = Hashtbl.create 31
+let code_uses : (location, code_dag_node) Hashtbl.t = Hashtbl.create 31
+let code_stores : code_dag_node list = ref []
+let code_loads : code_dag_node list = ref []
+let code_checkbounds : code_dag_node list = ref []
 
 let clear_code_dag () =
   Hashtbl.clear code_results;
