--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/bytecomp/translmod.ml	2012-05-30 06:29:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/bytecomp!translmod.ml.INPUT	2020-02-10 14:06:11.029889889 -0800
@@ -340,10 +340,10 @@
                rebind_idents (pos + 1) (id :: newfields) ids) in
       Llet(Strict, mid, transl_module Tcoerce_none None modl,
            rebind_idents 0 fields ids)
+;;
 
 (* Update forward declaration in Translcore *)
-let _ =
-  Translcore.transl_module := transl_module
+Translcore.transl_module := transl_module ;;
 
 (* Compile an implementation *)
 
