--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/bytecomp/bytegen.ml	2012-02-18 09:21:42.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/bytecomp!bytegen.ml.INPUT	2020-02-10 13:47:38.701490326 -0800
@@ -267,7 +267,7 @@
     rec_vars: Ident.t list;             (* mutually recursive fn names *)
     rec_pos: int }                      (* rank in recursive definition *)
 
-let functions_to_compile  = (Stack.create () : function_to_compile Stack.t)
+let functions_to_compile : function_to_compile Stack.t  = Stack.create ()
 
 (* Name of current compilation unit (for debugging events) *)
 
