--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/bytecomp/bytelibrarian.ml	2012-03-07 09:50:17.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/bytecomp!bytelibrarian.ml.INPUT	2020-02-10 13:52:07.399048709 -0800
@@ -67,7 +67,7 @@
     if buffer = cmo_magic_number then begin
       let compunit_pos = input_binary_int ic in
       seek_in ic compunit_pos;
-      let compunit = (input_value ic : compilation_unit) in
+      let compunit : compilation_unit = input_value ic in
       Bytelink.check_consistency ppf file_name compunit;
       copy_compunit ic oc compunit;
       close_in ic;
@@ -76,7 +76,7 @@
     if buffer = cma_magic_number then begin
       let toc_pos = input_binary_int ic in
       seek_in ic toc_pos;
-      let toc = (input_value ic : library) in
+      let toc : library = input_value ic in
       List.iter (Bytelink.check_consistency ppf file_name) toc.lib_units;
       add_ccobjs toc;
       List.iter (copy_compunit ic oc) toc.lib_units;
