--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/bytecomp/bytelink.ml	2012-04-16 08:27:42.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/bytecomp!bytelink.ml.INPUT	2020-02-10 13:50:03.314396176 -0800
@@ -123,7 +123,7 @@
          requires. *)
       let compunit_pos = input_binary_int ic in  (* Go to descriptor *)
       seek_in ic compunit_pos;
-      let compunit = (input_value ic : compilation_unit) in
+      let compunit : compilation_unit = input_value ic in
       close_in ic;
       List.iter add_required compunit.cu_reloc;
       Link_object(file_name, compunit) :: tolink
@@ -133,7 +133,7 @@
          in only if needed. *)
       let pos_toc = input_binary_int ic in    (* Go to table of contents *)
       seek_in ic pos_toc;
-      let toc = (input_value ic : library) in
+      let toc : library = input_value ic in
       close_in ic;
       add_ccobjs toc;
       let required =
