--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/source.ml	2012-05-30 06:29:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!source.ml.INPUT	2020-02-10 14:10:15.810471225 -0800
@@ -82,8 +82,7 @@
         (list_truncate !buffer_max_count ((mdle, buffer)::!buffer_list));
       buffer
 
-let buffer_content =
-  (fst : buffer -> string)
+let buffer_content : buffer -> string = fst
 
 let buffer_length x =
   String.length (buffer_content x)
