--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/program_loading.ml	2012-04-13 05:43:24.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!program_loading.ml.INPUT	2020-02-11 10:49:39.639675585 -0800
@@ -27,7 +27,7 @@
 (*** Load a program. ***)
 
 (* Function used for launching the program. *)
-let launching_func = ref (function () -> ())
+let launching_func = ref (fun () -> ())
 
 let load_program () =
   !launching_func ();
@@ -69,7 +69,7 @@
   Buffer.contents res
 
 (* A generic function for launching the program *)
-let generic_exec_unix cmdline = function () ->
+let generic_exec_unix cmdline = fun () ->
   if !debug_loading then
     prerr_endline "Launching program...";
   let child =
@@ -96,7 +96,7 @@
        (_, WEXITED 0) -> ()
      | _ -> raise Toplevel
 
-let generic_exec_win cmdline = function () ->
+let generic_exec_win cmdline = fun () ->
   if !debug_loading then
     prerr_endline "Launching program...";
   try ignore(create_process "cmd.exe" [| "/C"; cmdline() |] stdin stdout stderr)
@@ -112,7 +112,7 @@
 (* Execute the program by calling the runtime explicitly *)
 let exec_with_runtime =
   generic_exec
-    (function () ->
+    (fun () ->
       match Sys.os_type with
         "Win32" ->
           (* This fould fail on a file name with spaces
@@ -135,7 +135,7 @@
 (* Excute the program directly *)
 let exec_direct =
   generic_exec
-    (function () ->
+    (fun () ->
       match Sys.os_type with
         "Win32" ->
           (* See the comment above *)
@@ -152,7 +152,7 @@
 
 (* Ask the user. *)
 let exec_manual =
-  function () ->
+  fun () ->
     print_newline ();
     print_string "Waiting for connection...";
     print_string ("(the socket is " ^ !socket_name ^ ")");
@@ -169,12 +169,11 @@
 
 let set_launching_function func =
   launching_func := func
-
+;;
 (* Initialization *)
 
-let _ =
   set_launching_function exec_direct
-
+;;
 (*** Connection. ***)
 
 let connection = ref Primitives.std_io
