--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/time_travel.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!time_travel.ml.INPUT	2020-02-11 10:52:55.701675585 -0800
@@ -74,10 +74,10 @@
 let wait_for_connection checkpoint =
   try
     Exec.unprotect
-      (function () ->
+      (fun () ->
          let old_controller = Input_handling.current_controller !connection in
            execute_with_other_controller
-             (function
+             (fun
                 fd ->
                   old_controller fd;
                   if checkpoint.c_valid = true then
@@ -293,7 +293,7 @@
     Some {rep_type = Exited | Uncaught_exc} -> ()
   | _ ->
       Exec.protect
-        (function () ->
+        (fun () ->
            if !make_checkpoints then
              duplicate_current_checkpoint ()
            else
@@ -328,7 +328,7 @@
         if !debug_time_travel then begin
           print_string "Checkpoints : pid(time)"; print_newline ();
           List.iter
-            (function {c_time = time; c_pid = pid; c_valid = valid} ->
+            (fun {c_time = time; c_pid = pid; c_valid = valid} ->
               Printf.printf "%d(%Ld)%s " pid time
                             (if valid then "" else "(invalid)"))
             !checkpoints;
@@ -384,7 +384,7 @@
 (* --- Assume that the checkpoint is valid. *)
 let forget_process fd pid =
   let checkpoint =
-    List.find (function c -> c.c_pid = pid) (!current_checkpoint::!checkpoints)
+    List.find (fun c -> c.c_pid = pid) (!current_checkpoint::!checkpoints)
   in
     Printf.eprintf "Lost connection with process %d" pid;
     let kont =
@@ -439,7 +439,7 @@
 let internal_go_to time =
   let duration = time -- (current_time ()) in
     if duration > _0 then
-      execute_without_breakpoints (function () -> step_forward duration)
+      execute_without_breakpoints (fun () -> step_forward duration)
 
 (* Move to a given time. *)
 let go_to time =
