--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/trap_barrier.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!trap_barrier.ml.INPUT	2020-02-11 10:53:39.785675585 -0800
@@ -30,7 +30,7 @@
 let update_trap_barrier () =
   if !current_checkpoint.c_trap_barrier <> !current_trap_barrier then
     Exec.protect
-      (function () ->
+      (fun () ->
          set_trap_barrier !current_trap_barrier;
          !current_checkpoint.c_trap_barrier <- !current_trap_barrier)
 
