--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/input_handling.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!input_handling.ml.INPUT	2020-02-11 10:45:22.089675585 -0800
@@ -73,7 +73,7 @@
             select (List.map fst !active_files) [] [] (-1.)
           in
             List.iter
-              (function fd ->
+              (fun fd ->
                  let (funct, iochan) = (List.assoc fd !active_files) in
                    funct iochan)
               input
