--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/symbols.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!symbols.ml.INPUT	2020-02-10 14:11:24.773785119 -0800
@@ -19,17 +19,17 @@
 open Debugger_config (* Toplevel *)
 open Program_loading
 
-let modules =
-  ref ([] : string list)
+let modules : string list =
+  ref []
 
-let events =
-  ref ([] : debug_event list)
-let events_by_pc =
-  (Hashtbl.create 257 : (int, debug_event) Hashtbl.t)
-let events_by_module =
-  (Hashtbl.create 17 : (string, debug_event array) Hashtbl.t)
-let all_events_by_module =
-  (Hashtbl.create 17 : (string, debug_event list) Hashtbl.t)
+let events : debug_event list =
+  ref []
+let events_by_pc : (int, debug_event) Hashtbl.t =
+  Hashtbl.create 257
+let events_by_module : (string, debug_event array) Hashtbl.t =
+  Hashtbl.create 17
+let all_events_by_module : (string, debug_event list) Hashtbl.t =
+  Hashtbl.create 17
 
 let relocate_event orig ev =
   ev.ev_pos <- orig + ev.ev_pos;
@@ -57,7 +57,7 @@
   let eventlists = ref [] in
   for i = 1 to num_eventlists do
     let orig = input_binary_int ic in
-    let evl = (input_value ic : debug_event list) in
+    let evl : debug_event list = input_value ic in
     (* Relocate events in event list *)
     List.iter (relocate_event orig) evl;
     eventlists := evl :: !eventlists
