--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/program_management.ml	2012-05-30 06:29:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!program_management.ml.INPUT	2020-02-11 10:50:15.750675585 -0800
@@ -129,7 +129,7 @@
   if !debug_loading then
     prerr_endline "Opening a socket...";
   open_connection !socket_name
-    (function () ->
+    (fun () ->
       go_to _0;
       Symbols.set_all_events();
       exit_main_loop ())
