--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/debugger/show_information.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/debugger!show_information.ml.INPUT	2020-02-11 10:51:32.145675585 -0800
@@ -49,7 +49,7 @@
              fprintf ppf "Breakpoints : %a@."
              (fun ppf l ->
                List.iter
-                (function x -> fprintf ppf "%i " x) l)
+                (fun x -> fprintf ppf "%i " x) l)
              (List.sort compare breakpoints));
         show_point ev true
   | Some {rep_type = Exited} ->
@@ -94,7 +94,7 @@
       | breakpoints ->
           fprintf ppf "Breakpoints : %a@."
           (fun ppf l ->
-            List.iter (function x -> fprintf ppf "%i " x) l)
+            List.iter (fun x -> fprintf ppf "%i " x) l)
           (List.sort compare breakpoints);
       end;
       show_point sel_ev selected
