--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/lex/compact.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/lex!compact.ml.INPUT	2020-02-10 14:12:49.352932810 -0800
@@ -47,9 +47,9 @@
       let r = do_emit_code c in
       Hashtbl.add memory c r ;
       r
-
+;;
 (* Code address 0 is the empty code (ie do nothing) *)
-let _ = mem_emit_code []
+mem_emit_code [] ;;
 
 let emit_tag_code c = mem_emit_code (List.fold_right ins_tag c [])
 and emit_mem_code c  =mem_emit_code (List.fold_right ins_mem c [])
