--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_latex.ml	2012-03-20 05:00:11.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_latex.ml.INPUT	2020-02-11 13:50:47.725714265 -0800
@@ -291,8 +291,6 @@
     method latex_of_Target fmt ~target ~code =
       if String.lowercase target = "latex" then
         self#latex_of_Latex fmt code
-      else
-        ()
 
     method latex_of_Raw fmt s =
       ps fmt (self#escape s)
