--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_man.ml	2012-03-20 05:00:11.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_man.ml.INPUT	2020-02-11 13:51:00.145714265 -0800
@@ -315,7 +315,7 @@
     method man_of_custom_text b s t = ()
 
     method man_of_Target b ~target ~code =
-      if String.lowercase target = "man" then bs b code else ()
+      if String.lowercase target = "man" then bs b code
 
     (** Print groff string to display code. *)
     method man_of_code b s = self#man_of_text b [ Code s ]
