--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamlbuild/findlib.ml	2012-07-16 01:53:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamlbuild!findlib.ml.INPUT	2020-02-11 13:06:35.877675585 -0800
@@ -61,7 +61,7 @@
 
 let run_and_parse lexer command =
   Printf.ksprintf
-    (fun command -> lexer & Lexing.from_string & run_and_read command)
+    (fun command -> lexer && Lexing.from_string && run_and_read command)
     command
 
 let run_and_read command =
@@ -129,7 +129,7 @@
   with Not_found -> s
 
 let list () =
-  List.map before_space (split_nl & run_and_read "%s list" ocamlfind)
+  List.map before_space (split_nl && run_and_read "%s list" ocamlfind)
 
 (* The closure algorithm is easy because the dependencies are already closed
 and sorted for each package. We only have to make the union. We could also
