--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_global.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_global.ml.INPUT	2020-02-11 13:49:49.151714265 -0800
@@ -14,7 +14,7 @@
 (** Global variables. *)
 
 (* Tell ocaml compiler not to generate files. *)
-let _ = Clflags.dont_write_files := true
+Clflags.dont_write_files := true ;;
 
 open Clflags
 
