--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamlbuild/glob.ml	2010-01-22 04:48:24.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/ocamlbuild!glob.ml.INPUT	2020-02-12 21:24:01.621859611 -0800
@@ -37,7 +37,7 @@
 (*** match_character_class *)
 let match_character_class cl c =
   Bool.eval
-    begin function (c1,c2) ->
+    begin fun (c1,c2) ->
       c1 <= c && c <= c2
     end
     cl
