--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_print.ml	2012-05-30 06:29:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_print.ml.INPUT	2020-02-11 13:51:48.510714265 -0800
@@ -25,7 +25,7 @@
   (fmt, flush)
 
 let (type_fmt, flush_type_fmt) = new_fmt ()
-let _ =
+;;
   let (out, flush, outnewline, outspace) =
     pp_get_all_formatter_output_functions type_fmt ()
   in
@@ -33,6 +33,7 @@
     ~out ~flush
     ~newline: (fun () -> out "\n  " 0 3)
     ~spaces: outspace
+;;
 
 let (modtype_fmt, flush_modtype_fmt) = new_fmt ()
 
