--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_dot.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_dot.ml.INPUT	2020-02-11 13:49:37.600714265 -0800
@@ -84,7 +84,7 @@
     method generate_for_module fmt m =
       let l = List.filter
           (fun n ->
-            !dot_include_all or
+            !dot_include_all ||
             (List.exists (fun m -> m.Module.m_name = n) modules))
           m.Module.m_top_deps
       in
