--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamlbuild/my_std.ml	2012-05-02 03:02:53.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamlbuild!my_std.ml.INPUT	2020-02-11 13:35:35.794714265 -0800
@@ -179,7 +179,7 @@
     and n = String.length v
     in
     m <= n &&
-      let rec loop i = i = m or u.[i] = v.[i] && loop (i + 1) in
+      let rec loop i = i = m || u.[i] = v.[i] && loop (i + 1) in
       loop 0
   (* ***)
 
@@ -189,7 +189,7 @@
     and n = String.length v
     in
     n <= m &&
-      let rec loop i = i = n or u.[m - 1 - i] = v.[n - 1 - i] && loop (i + 1) in
+      let rec loop i = i = n || u.[m - 1 - i] = v.[n - 1 - i] && loop (i + 1) in
       loop 0
   (* ***)
 
