--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamlbuild/plugin.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamlbuild!plugin.ml.INPUT	2020-02-11 13:56:18.496714265 -0800
@@ -50,8 +50,8 @@
 
     let rebuild_plugin_if_needed () =
       let a = up_to_date_or_copy plugin_file in
-      let b = (not we_have_a_config_file) or up_to_date_or_copy plugin_config_file in
-      let c = (not we_have_a_config_file_interface) or up_to_date_or_copy plugin_config_file_interface in
+      let b = (not we_have_a_config_file) || up_to_date_or_copy plugin_config_file in
+      let c = (not we_have_a_config_file_interface) || up_to_date_or_copy plugin_config_file_interface in
       if a && b && c && we_have_a_plugin then
         () (* Up to date *)
            (* FIXME: remove ocamlbuild_config.ml in _build/ if removed in parent *)
@@ -109,8 +109,6 @@
             let () = Log.finish () in
             raise (Exit_silently_with_code (sys_command (Command.string_of_command_spec spec)))
         end
-      else
-        ()
   end
 ;;
 
