--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamldoc/odoc_info.ml	2012-05-07 03:31:18.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamldoc!odoc_info.ml.INPUT	2020-02-11 13:50:36.732714265 -0800
@@ -175,8 +175,6 @@
 let verbose s =
   if !Odoc_global.verbose then
     (print_string s ; print_newline ())
-  else
-    ()
 
 let warning s = Odoc_global.pwarning s
 let print_warnings = Odoc_config.print_warnings
