--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/ocamlbuild/main.ml	2012-05-15 04:28:57.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/ocamlbuild!main.ml.INPUT	2020-02-11 13:35:02.724714265 -0800
@@ -208,7 +208,6 @@
       end targets [] in
 
     if !Options.program_to_execute then
-      begin
         match List.rev cmds with
         | [] -> raise (Exit_usage "Using -- requires one target");
         | cmd :: rest ->
@@ -216,9 +215,6 @@
           let cmd_spec = S [P cmd; atomize !Options.program_args] in
           dprintf 3 "Running the user command:@ %a" Pathname.print cmd;
           raise (Exit_with_code (call cmd_spec)) (* Exit with the exit code of the called command *)
-      end
-    else
-      ()
   with
   | Ocaml_dependencies.Circular_dependencies(seen, p) ->
       raise
