--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/examples_camltk/jptest.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!examples_camltk!jptest.ml.INPUT	2020-02-11 15:57:49.992714265 -0800
@@ -18,6 +18,6 @@
 let win = opentk();;
 
 let b = Button.create win [ Text "こんちは" ];;
-let _ = pack [b] [];;
+pack [b] [];;
 
 mainLoop();;
