--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/examples_labltk/clock.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!examples_labltk!clock.ml.INPUT	2020-02-11 16:41:13.372372703 -0800
@@ -128,6 +128,7 @@
 (* Create a clock on the main window *)
 let clock =
   new clock ~parent:top
-
+;;
 (* Wait for events *)
-let _ = mainLoop ()
+mainLoop ()
+;;
