--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/unix/unix.ml	2012-01-18 01:28:05.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/otherlibs!unix!unix.ml.INPUT	2020-02-11 17:24:18.974259431 -0800
@@ -85,11 +85,14 @@
   | EUNKNOWNERR of int
 
 exception Unix_error of error * string * string
+;;
 
-let _ = Callback.register_exception "Unix.Unix_error"
+Callback.register_exception "Unix.Unix_error"
                                     (Unix_error(E2BIG, "", ""))
+;;
 
 external error_message : error -> string = "unix_error_message"
+;;
 
 let handle_unix_error f arg =
   try
@@ -823,7 +826,7 @@
   | Process_out of out_channel
   | Process_full of in_channel * out_channel * in_channel
 
-let popen_processes = (Hashtbl.create 7 : (popen_process, int) Hashtbl.t)
+let popen_processes : (popen_process, int) Hashtbl.t = Hashtbl.create 7
 
 let open_proc cmd proc input output toclose =
   let cloexec = List.for_all try_set_close_on_exec toclose in
