--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/compiler/tsort.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!compiler!tsort.ml.INPUT	2020-02-11 14:37:22.935714265 -0800
@@ -65,7 +65,7 @@
     let q = Queue.create ()
     and result = ref [] in
     List.iter !order
-      ~f:(function {pred_count = n} as node ->
+      ~f:(fun ({pred_count = n} as node) ->
                 if n = 0 then Queue.add node q);
     begin try
       while true do
