--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/browser/main.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!browser!main.ml.INPUT	2020-02-11 13:55:41.197714265 -0800
@@ -61,10 +61,10 @@
 let usage ~spec errmsg =
   let b = Buffer.create 1024 in
   bprintf b "%s\n" errmsg;
-  List.iter (function (key, _, doc) -> bprintf b "  %s %s\n" key doc) spec;
+  List.iter (fun (key, _, doc) -> bprintf b "  %s %s\n" key doc) spec;
   Buffer.contents b
+;;
 
-let _ =
   let is_win32 = Sys.os_type = "Win32" in
   if is_win32 then
     Format.pp_set_formatter_output_functions Format.err_formatter
@@ -130,3 +130,4 @@
     with Protocol.TkError _ ->
       if not is_win32 then flush stderr
   done
+;;
