--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/examples_camltk/text.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!examples_camltk!text.ml.INPUT	2020-02-11 16:40:16.073990280 -0800
@@ -37,15 +37,14 @@
 
 let yank () =
   Text.insert text (TextIndex (Insert, [])) !buffer []
-
-let _ = bind text [[Control], KeyPressDetail "y"] (BindSet ([], fun _ ->
+;;
+bind text [[Control], KeyPressDetail "y"] (BindSet ([], fun _ ->
   yank () ))
 ;;
-let _ = bind text [[Control], KeyPressDetail "k"] (BindSet ([], fun _ ->
+bind text [[Control], KeyPressDetail "k"] (BindSet ([], fun _ ->
   kill () ))
 ;;
 
-let _ =
   scroll_link scrollbar text;
 
   pack [text;f][];
