--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/frx/frx_synth.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!frx!frx_synth.ml.INPUT	2020-02-11 17:07:11.719593927 -0800
@@ -83,6 +83,7 @@
 (* Remove all callbacks associated to widget <w> *)
 let remove_callbacks w =
   Hashtbl.iter (fun _ h -> Hashtbl.remove h (Widget.name w)) events
+;;
 
-let _ =
   add_destroy_hook remove_callbacks
+;;
