--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/threads/pervasives.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!threads!pervasives.ml.INPUT	2020-02-11 17:20:03.104319582 -0800
@@ -544,5 +544,7 @@
 
 external register_named_value : string -> 'a -> unit
                               = "caml_register_named_value"
+;;
 
-let _ = register_named_value "Pervasives.do_at_exit" do_at_exit
+register_named_value "Pervasives.do_at_exit" do_at_exit
+;;
