--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/examples_labltk/tetris.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!examples_labltk!tetris.ml.INPUT	2020-02-11 16:56:26.658908863 -0800
@@ -366,7 +366,7 @@
     done
   done
 
-let timer_ref = (ref None : Timer.t option ref)
+let timer_ref : Timer.t option ref = ref None
 (* I know, this should be timer ref, but I'm not sure what should be
    the initial value ... *)
 
@@ -385,8 +385,8 @@
     y= !c.y;
     d= !c.d;
     alive= !c.alive }
+;;
 
-let _ =
   let top = openTk () in
   let lb = Label.create top
   and fw = Frame.create top
@@ -706,5 +706,6 @@
   bind_game top;
   Button.configure button ~command: game_init;
   game_init ()
+;;
 
-let _ = Printexc.print mainLoop ()
+Printexc.print mainLoop () ;;
