--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/examples_camltk/helloworld.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!examples_camltk!helloworld.ml.INPUT	2020-02-11 15:57:18.161714265 -0800
@@ -21,7 +21,7 @@
 (* Widget initialisation *)
 let b = Button.create top
           [Text "foobar";
-           Command (function () ->
+           Command (fun () ->
                       print_string "foobar";
                       print_newline();
                       flush stdout)];;
