--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/num/nat.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!num!nat.ml.INPUT	2020-02-11 17:18:48.765835565 -0800
@@ -49,7 +49,9 @@
 external lxor_digit_nat: nat -> int -> nat -> int -> unit = "lxor_digit_nat"
 
 external initialize_nat: unit -> unit = "initialize_nat"
-let _ = initialize_nat()
+;;
+
+initialize_nat() ;;
 
 let length_nat (n : nat) = Obj.size (Obj.repr n) - 1
 
