--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/browser/searchpos.ml	2012-07-10 01:33:16.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!browser!searchpos.ml.INPUT	2020-02-11 14:06:41.148714265 -0800
@@ -662,7 +662,7 @@
 
 let rec search_pos_structure ~pos str =
   List.iter str ~f:
-  begin function str -> match str.str_desc with
+  begin fun str -> match str.str_desc with
     Tstr_eval exp -> search_pos_expr exp ~pos
   | Tstr_value (rec_flag, l) ->
       List.iter l ~f:
@@ -689,7 +689,7 @@
 
 and search_pos_class_structure ~pos cls =
   List.iter cls.cstr_fields ~f:
-    begin function cf -> match cf.cf_desc with
+    begin fun cf -> match cf.cf_desc with
         Tcf_inher (_, cl, _, _, _) ->
           search_pos_class_expr cl ~pos
       | Tcf_val (_, _, _, _, Tcfk_concrete exp, _) -> search_pos_expr exp ~pos
