--- /home/chetsky/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/dynlink/extract_crc.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!dynlink!extract_crc.ml.INPUT	2020-02-11 13:54:01.553714265 -0800
@@ -49,5 +49,7 @@
            "<dir>  Add <dir> to the list of include directories"]
     print_crc usage;
   print_string "\n]\n"
+;;
 
-let _ = main(); exit 0
+main(); exit 0
+;;
