--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/otherlibs/labltk/frx/frx_misc.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/otherlibs!labltk!frx!frx_misc.ml.INPUT	2020-02-11 17:07:00.428324336 -0800
@@ -16,7 +16,7 @@
 (* Delayed global, a.k.a cache&carry *)
 let autodef f =
   let v = ref None in
-  (function () ->
+  (fun () ->
      match !v with
        None ->
          let x = f() in
