--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/stdlib/camlinternalOO.ml	2011-12-21 23:30:18.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/stdlib!camlinternalOO.ml.INPUT	2020-02-11 17:26:46.987129248 -0800
@@ -59,7 +59,7 @@
 
 type item = DummyA | DummyB | DummyC of int
 
-let dummy_item = (magic () : item)
+let dummy_item : item = magic ()
 
 (**** Types ****)
 
