--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/stdlib/pervasives.ml	2012-01-12 07:46:51.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/stdlib!pervasives.ml.INPUT	2020-02-11 17:27:17.565401754 -0800
@@ -453,5 +453,6 @@
 
 external register_named_value : string -> 'a -> unit
                               = "caml_register_named_value"
+;;
 
-let _ = register_named_value "Pervasives.do_at_exit" do_at_exit
+register_named_value "Pervasives.do_at_exit" do_at_exit ;;
