--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/gc-roots/globroots.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!gc-roots!globroots.ml.INPUT	2020-02-11 17:39:46.566726856 -0800
@@ -68,8 +68,8 @@
 
 module TestClassic = Test(Classic)
 module TestGenerational = Test(Generational)
+;;
 
-let _ =
   let n =
     if Array.length Sys.argv < 2 then 10000 else int_of_string Sys.argv.(1) in
   print_string "Non-generational API\n";
@@ -78,3 +78,5 @@
   print_string "Generational API\n";
   TestGenerational.test n;
   print_newline()
+;;
+
