--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/torture.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!torture.ml.INPUT	2020-02-11 20:16:23.980445423 -0800
@@ -41,5 +41,6 @@
   Thread.create writer_thread (in2, 16);
   Thread.create reader_thread (out2, 16);
   stdin_thread()
+;;
 
-let _ = main()
+main() ;;
