--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/misc/taku.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!misc!taku.ml.INPUT	2020-02-11 20:28:09.676733434 -0800
@@ -18,5 +18,6 @@
 
 let rec repeat n =
   if n <= 0 then 0 else tak(18,12,6) + repeat(n-1)
+;;
 
-let _ = print_int (repeat 2000); print_newline(); exit 0
+print_int (repeat 2000); print_newline(); exit 0 ;;
