--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/typing-objects/Tests.ml	2011-12-21 23:30:18.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!typing-objects!Tests.ml.INPUT	2020-02-11 20:53:50.911660528 -0800
@@ -270,7 +270,7 @@
 
 let x = ref [];;
 module F(X : sig end) =
-  struct type t = int let _ = (x : < m : t> list ref) end;;
+  struct type t = int ;; (x : < m : t> list ref) ;; end;;
 x;;
 
 type 'a t;;
