--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-digest/md5.ml	2012-02-10 08:15:24.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-digest!md5.ml.INPUT	2020-02-11 17:41:58.486242506 -0800
@@ -197,8 +197,8 @@
   for i = 1 to iter do fn () done;
   let stop = Sys.time() in
   printf "%s: %.2f s\n" msg (stop -. start)
+;;
 
-let _ =
   (* Test *)
   if test ""
   && test "a"
@@ -219,3 +219,4 @@
     time "C implementation" num_iter
       (fun () -> ignore (Digest.string s))
   end
+;;
