--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/basic/tailcalls.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!basic!tailcalls.ml.INPUT	2020-02-11 17:38:37.829020765 -0800
@@ -19,10 +19,11 @@
 
 let indtailcall16 fn a b c d e f g h i j k l m n o p =
   fn a b c d e f g h i j k l m n o p 
+;;
 
-let _ =
   print_int (tailcall4 10000000 0 0 0); print_newline();
   print_int (tailcall8 10000000 0 0 0 0 0 0 0); print_newline();
   print_int (tailcall16 10000000 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0); print_newline();
   print_int (indtailcall8 tailcall8 10 0 0 0 0 0 0 0); print_newline();
   print_int (indtailcall16 tailcall16 10 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0); print_newline()
+;;
