--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/test6.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!test6.ml.INPUT	2020-02-11 20:14:23.135416043 -0800
@@ -1,6 +1,6 @@
 open Event
 
-let ch = (new_channel() : string channel)
+let ch : string channel = new_channel()
 
 let rec f tag msg =
   select [
@@ -8,8 +8,9 @@
     wrap (receive ch) (fun x -> print_string(tag ^ ": " ^ x); print_newline())
   ];
   f tag msg
+;;
 
-let _ =
   Thread.create (f "A") "hello";
   f "B" "world";
   exit 0
+;;
