--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/token1.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!token1.ml.INPUT	2020-02-11 20:16:23.996445551 -0800
@@ -32,5 +32,6 @@
   niter := iter;
   for i = 0 to nprocs - 1 do Thread.create process (i, conds, nprocs) done;
   Thread.delay 3600.
+;;
 
-let _ = main()
+main() ;;
