--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/testsignal.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!testsignal.ml.INPUT	2020-02-11 20:16:24.008445647 -0800
@@ -6,8 +6,9 @@
   while true do
     print_char c; flush stdout; Thread.delay delay
   done
+;;
 
-let _ =
   Sys.signal Sys.sigint (Sys.Signal_handle sighandler);
   Thread.create (print_message 0.6666666666) 'a';
   print_message 1.0 'b'
+;;
