--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/basic-io-2/io.ml	2012-06-04 05:18:53.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!basic-io-2!io.ml.INPUT	2020-02-11 17:30:33.391671929 -0800
@@ -78,10 +78,9 @@
     output_string oc (String.make (i*64) '.'); output_char oc '\n'
   done;
   close_out oc
-
+;;
 (* The test *)
 
-let _ =
   let src = Sys.argv.(1) in
   let testio = Filename.temp_file "testio" "" in
   let lines = Filename.temp_file "lines" "" in
@@ -101,3 +100,4 @@
   Sys.remove lines;
   Sys.remove testio;
   exit 0
+;;
