--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/backtrace/backtrace.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!backtrace!backtrace.ml.INPUT	2020-02-11 17:29:49.456416346 -0800
@@ -11,7 +11,8 @@
   with Error "a" -> print_string "a"; print_newline(); 0
      | Error "b" as exn -> print_string "b"; print_newline(); raise exn
      | Error "c" -> raise (Error "c")
+;;
 
-let _ =
   Printexc.record_backtrace true;
   ignore (g Sys.argv.(1))
+;;
