--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/misc-unsafe/almabench.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!misc-unsafe!almabench.ml.INPUT	2020-02-11 20:21:33.286593107 -0800
@@ -235,7 +235,7 @@
     let dae = ref ((am -. !ae +. de *. sin !ae) /. (1.0 -. de *. cos !ae)) in
       ae := !ae +. !dae;
       incr k;
-      while !k < 10 or abs_float !dae >= 1e-12 do
+      while !k < 10 || abs_float !dae >= 1e-12 do
 	dae := (am -. !ae +. de *. sin !ae) /. (1.0 -. de *. cos !ae);
 	ae := !ae +. !dae;
 	incr k
@@ -294,11 +294,11 @@
   
   (* Declination *)
   rdd.(1) <- asin (state.(0).(2) /. rdd.(2)) *. r2d
-  
+;;  
 
 
 (* Entry point. Calculate RA and Dec for noon on every day in 1900-2100 *)
-let _ =
+
   let jd = [| 0.0; 0.0 |]
   and pv = [| [| 0.0; 0.0; 0.0 |]; [| 0.0; 0.0; 0.0 |] |] 
   and position = [| 0.0; 0.0; 0.0 |] in
@@ -322,3 +322,4 @@
       done
     done
   done
+;;
