--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/basic/arrays.ml	2011-12-21 02:36:35.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!basic!arrays.ml.INPUT	2020-02-11 17:32:41.095198726 -0800
@@ -109,8 +109,8 @@
   Array.blit a 5 a 6 4;
   if a <> [|"a"; "a"; "a"; "a"; "a"; "b1"; "b1"; "b2"; "b3"; "a"|]
   then print_string "Test7: failed(2)\n"
+;;
 
-let _ =
   test1();
   test2();
   test3();
@@ -119,3 +119,4 @@
   test6();
   test7();
   exit 0
+;;
