--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/basic/recvalues.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!basic!recvalues.ml.INPUT	2020-02-11 17:37:53.699987415 -0800
@@ -1,6 +1,5 @@
 (* Recursive value definitions *)
 
-let _ =
   let rec x = 1 :: x in
   if match x with
        1 :: x' -> x == x'
@@ -33,6 +32,6 @@
 ;;
 
 Gc.minor ();;
-if (List.hd (!(snd (x2.(0))))) () == s
+if List.hd (!(snd (x2.(0)))) () == s
 then print_string "Test 4: passed\n"
 else print_string "Test 4: FAILED\n"
