--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/testio.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!testio.ml.INPUT	2020-02-11 20:16:24.024445775 -0800
@@ -116,5 +116,6 @@
   Sys.remove linesfile;
   Sys.remove ofile;
   exit 0
+;;
 
-let _ = Unix.handle_unix_error main (); exit 0
+Unix.handle_unix_error main (); exit 0 ;;
