--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/test8.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!test8.ml.INPUT	2020-02-11 20:14:56.923717605 -0800
@@ -38,7 +38,8 @@
   print_int (sync(receive ch)); print_newline();
   print_string (sync(buffer_receive box)); print_newline();
   print_string (sync(buffer_receive box)); print_newline()
+;;
 
-let _ =
   Thread.create f ();
   g()
+;;
