--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/misc/sieve.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!misc!sieve.ml.INPUT	2020-02-11 20:26:06.612117996 -0800
@@ -48,9 +48,9 @@
 let rec do_list f = function
     [] -> ()
   | a::l -> f a; do_list f l
+;;
 
-
-let _ =
   do_list (fun n -> print_int n; print_string " ") (sieve 50000);
   print_newline();
   exit 0
+;;
