--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/basic/patmatch.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!basic!patmatch.ml.INPUT	2020-02-11 17:37:17.047173800 -0800
@@ -79,8 +79,8 @@
       string_unsafe_set s 3 (unsafe_chr (48 + n mod 10));
       s
     end
+;;
 
-let _ =
   for i = -5 to 10 do printf "f(%d) = %d\n" i (f i) done;
   List.iter (fun i -> printf "g(%d) = %d\n" i (g i))
             [0;300;303;305;400;401;402;999];
@@ -103,6 +103,7 @@
   printf "l([|2;3|]) = %d\n" (l [|2;3|]);
   printf "l([|4;5;6|]) = %d\n" (l [|4;5;6|]);
   exit 0
+;;
 
 
 
