--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/typing-objects-bugs/yamagata021012_ok.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!typing-objects-bugs!yamagata021012_ok.ml.INPUT	2020-02-11 20:52:55.928097387 -0800
@@ -39,7 +39,7 @@
     method iter proc =
       let p = self#first in
       for i = 0 to self#len - 2 do proc p#get; p#incr () done;
-      if self#len > 0 then proc p#get else ()
+      if self#len > 0 then proc p#get
   end
 
 class type ['a] obj_input_channel =
