--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/prim-revapply/revapply.ml	2012-01-23 06:49:39.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!prim-revapply!revapply.ml.INPUT	2020-02-11 20:29:18.541063731 -0800
@@ -4,8 +4,8 @@
 let g x = x * x
 let h x = x + 1
 let add x y = x + y
+;;
 
-let _ =
   List.iter (fun x ->
     print_int x; print_newline ()
   )
@@ -16,3 +16,4 @@
       3 |> f |> g |> h; (* 37 *)
       3 |> add 2 |> add 3 |> f |> g |> add 4; (* 260 *)
     ]
+;;
