--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/prim-revapply/apply.ml	2012-01-23 06:49:39.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!prim-revapply!apply.ml.INPUT	2020-02-11 20:29:51.401218268 -0800
@@ -4,8 +4,8 @@
 let g x = x * x
 let h x = x + 1
 let add x y = x + y
+;;
 
-let _ =
   List.iter (fun x ->
     print_int x; print_newline ()
   )
@@ -16,14 +16,16 @@
       h @@ g @@ f @@ 3; (* 37 *)
       add 4 @@ g @@ f @@ add 3 @@ add 2 @@ 3; (* 260 *)
     ]
+;;
+
 external ( @@ ) :  ('a -> 'b) -> 'a -> 'b = "%apply"
 
 let f x = x + x
 let g x = x * x
 let h x = x + 1
 let add x y = x + y
+;;
 
-let _ =
   List.iter (fun x ->
     print_int x; print_newline ()
   )
@@ -34,3 +36,4 @@
       h @@ g @@ f @@ 3; (* 37 *)
       add 4 @@ g @@ f @@ add 3 @@ add 2 @@ 3; (* 260 *)
     ]
+;;
