--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/test7.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!test7.ml.INPUT	2020-02-11 20:14:39.291561659 -0800
@@ -22,5 +22,6 @@
   Thread.create (sender add_ch) 1;
   Thread.create (sender sub_ch) 1;
   while true do read() done
+;;
 
-let _ = Printexc.catch main ()
+Printexc.catch main () ;;
