--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/tool-lexyacc/main.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!tool-lexyacc!main.ml.INPUT	2020-02-11 20:31:34.837693380 -0800
@@ -55,9 +55,9 @@
   output_lexdef header dfa;
   close_in !ic;
   close_out !oc
-
-let _ = main(); exit 0
-
+;;
+main(); exit 0
+;;
 
 (*****
 let main () =
