--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/test2.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!test2.ml.INPUT	2020-02-11 20:13:05.966681791 -0800
@@ -5,11 +5,13 @@
     print_char c; flush stdout;
     if !yield then Thread.yield()
   done
+;;
+
+yield := (Array.length Sys.argv > 1) ;;
 
-let _ = yield := (Array.length Sys.argv > 1)
 let t1 = Thread.create print_message 'a'
 let t2 = Thread.create print_message 'b'
-let _ = Thread.join t1
-let _ = Thread.join t2
-
 ;;
+
+Thread.join t1 ;;
+Thread.join t2 ;;
