--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/runtime-errors/stackoverflow.ml	2011-12-20 06:38:53.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!runtime-errors!stackoverflow.ml.INPUT	2020-02-11 20:30:09.769303851 -0800
@@ -7,9 +7,10 @@
     with Stack_overflow ->
       print_string "x = "; print_int x; print_newline();
       raise Stack_overflow
+;;
 
-let _ =
   try
     ignore(f 0)
   with Stack_overflow ->
     print_string "Stack overflow caught"; print_newline()
+;;
