--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/misc-unsafe/soli.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!misc-unsafe!soli.ml.INPUT	2020-02-11 20:22:27.226915469 -0800
@@ -106,6 +106,6 @@
       false
     with Found ->
       true
+;;
 
-
-let _ = if solve 0 then (print_string "\n"; print_board board)
+if solve 0 then (print_string "\n"; print_board board) ;;
