--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/test5.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!test5.ml.INPUT	2020-02-11 20:14:03.231232927 -0800
@@ -1,6 +1,6 @@
 open Event
 
-let ch = (new_channel() : string channel)
+let ch : string channel = new_channel()
 
 let rec sender msg =
   sync (send ch msg);
@@ -10,10 +10,11 @@
   print_string (name ^ ": " ^ sync (receive ch) ^ "\n");
   flush stdout;
   receiver name
+;;
 
-let _ =
   Thread.create sender "hello";
   Thread.create sender "world";
   Thread.create receiver "A";
   receiver "B";
   exit 0
+;;
