--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-set/testset.ml	2012-01-16 01:03:16.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-set!testset.ml.INPUT	2020-02-11 20:11:07.697413508 -0800
@@ -113,8 +113,8 @@
   let s = ref S.empty in
   for i = 1 to Random.int 10 do s := S.add (relt()) !s done;
   !s
+;;
 
-let _ =
   Random.init 42;
   for i = 1 to 25000 do test (relt()) (rset()) (rset()) done
-  
+;;  
