--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/tool-ocaml/t330-compact-2.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!tool-ocaml!t330-compact-2.ml.INPUT	2020-02-11 20:32:16.349879744 -0800
@@ -1,6 +1,6 @@
 open Lib;;
 Gc.compact ();;
-let _ = Pervasives.do_at_exit();;
+Pervasives.do_at_exit();;
 
 (**
        0 CONSTINT 42
