--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-threads/testexit.ml	2011-07-20 02:17:07.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-threads!testexit.ml.INPUT	2020-02-11 20:16:24.032445839 -0800
@@ -13,9 +13,10 @@
     flush stdout;
     Thread.delay 0.5
   done
+;;
 
-let _ =
   let _ = Thread.create somethread ("A", 5, false) in
   let _ = Thread.create somethread ("B", 8, false) in
   let _ = Thread.create somethread ("C", 11, true) in
   somethread ("Main", 3, false)
+;;
