--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/testsuite/tests/lib-dynlink-native/main.ml	2012-03-13 07:44:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/testsuite!tests!lib-dynlink-native!main.ml.INPUT	2020-02-11 17:42:56.675865167 -0800
@@ -25,7 +25,7 @@
     Marshal.to_channel oc !Api.cbs [Marshal.Closures];
     close_out oc;
     let ic = open_in_bin "marshal.data" in
-    let l = (Marshal.from_channel ic : (unit -> unit) list) in
+    let l : (unit -> unit) list = Marshal.from_channel ic in
     close_in ic;
     List.iter (fun f -> f()) l
   with Failure s ->
