--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/tools/ocamlprof.ml	2012-05-30 06:29:48.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/tools!ocamlprof.ml.INPUT	2020-02-11 21:11:06.334867074 -0800
@@ -367,7 +367,7 @@
   match item.pstr_desc with
     Pstr_eval exp -> rewrite_exp iflag exp
   | Pstr_value(_, exps)
-     -> List.iter (function (_,exp) -> rewrite_exp iflag exp) exps
+     -> List.iter (fun (_,exp) -> rewrite_exp iflag exp) exps
   | Pstr_module(name, smod) -> rewrite_mod iflag smod
   | Pstr_class classes -> List.iter (rewrite_class_declaration iflag) classes
   | _ -> ()
@@ -424,8 +424,8 @@
    end else begin
      (* Results mode *)
      let ic = open_in_bin !dumpfile in
-     let allcounters =
-       (input_value ic : (string * (string * int array)) list) in
+     let allcounters : (string * (string * int array)) list =
+       input_value ic in
      close_in ic;
      let (modes, cv) =
        try
@@ -498,5 +498,6 @@
     | x -> raise x in
     report_error Format.err_formatter x;
     exit 2
+;;
 
-let _ = main ()
+main () ;;
