--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/tools/ocamlmktop.ml	2012-05-24 09:17:19.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/tools!ocamlmktop.ml.INPUT	2020-02-11 21:10:19.562653953 -0800
@@ -12,6 +12,6 @@
 
 (* $Id: ocamlmktop.ml 12477 2012-05-24 16:17:19Z xleroy $ *)
 
-let _ =
   let args = Ccomp.quote_files (List.tl (Array.to_list Sys.argv)) in
   exit(Sys.command("ocamlc -I +compiler-libs -linkall ocamlcommon.cma ocamlbytecomp.cma ocamltoplevel.cma " ^ args ^ " topstart.cmo"))
+;;
