--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/tools/profiling.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/tools!profiling.ml.INPUT	2020-02-11 21:12:39.135493039 -0800
@@ -29,7 +29,7 @@
   in
   begin try
     let ic = open_in_bin dumpfile in
-    let prevl = (input_value ic : profiling_counters) in
+    let prevl : profiling_counters = input_value ic in
     close_in ic;
     List.iter2
       (fun (curname, (curmodes,curcount)) (prevname, (prevmodes,prevcount)) ->
@@ -52,5 +52,5 @@
     close_out oc
   with _ -> ()
   end
-
-let _ = at_exit dump_counters
+;;
+at_exit dump_counters ;;
