--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/tools/primreq.ml	2011-07-27 07:17:02.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/tools!primreq.ml.INPUT	2020-02-11 21:12:11.191319403 -0800
@@ -44,14 +44,14 @@
   if buffer = cmo_magic_number then begin
     let cu_pos = input_binary_int ic in
     seek_in ic cu_pos;
-    let cu = (input_value ic : compilation_unit) in
+    let cu : compilation_unit = input_value ic in
     close_in ic;
     scan_info cu
   end else
   if buffer = cma_magic_number then begin
     let toc_pos = input_binary_int ic in
     seek_in ic toc_pos;
-    let toc = (input_value ic : library) in
+    let toc : library = input_value ic in
     close_in ic;
     List.iter scan_info toc.lib_units
   end else begin
@@ -86,5 +86,6 @@
       if s.[0] <> '%' then begin print_string s; print_newline() end)
     !primitives;
   exit 0
+;;
 
-let _ = main ()
+main () ;;
