--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/tools/ocamldep.ml	2012-07-23 06:39:21.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/tools!ocamldep.ml.INPUT	2020-02-11 21:10:09.634690407 -0800
@@ -341,7 +341,6 @@
     file_dependencies_as ML source_file
   else if List.exists (Filename.check_suffix source_file) !mli_synonyms then
     file_dependencies_as MLI source_file
-  else ()
 
 let sort_files_by_dependencies files =
   let h = Hashtbl.create 31 in
@@ -430,7 +429,6 @@
   exit 0;
 ;;
 
-let _ =
   Clflags.classic := false;
   add_to_load_path Filename.current_dir_name;
   Arg.parse [
@@ -465,3 +463,4 @@
     ] file_dependencies usage;
   if !sort_files then sort_files_by_dependencies !files;
   exit (if !error_occurred then 2 else 0)
+;;
