--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/toplevel/toploop.ml	2012-07-10 07:54:19.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/toplevel!toploop.ml.INPUT	2020-02-11 21:20:40.141067365 -0800
@@ -33,8 +33,8 @@
 
 (* The table of toplevel value bindings and its accessors *)
 
-let toplevel_value_bindings =
-  (Hashtbl.create 37 : (string, Obj.t) Hashtbl.t)
+let toplevel_value_bindings : (string, Obj.t) Hashtbl.t =
+  Hashtbl.create 37
 
 let getvalue name =
   try
@@ -131,7 +131,7 @@
   Symtable.update_global_table();
   try
     may_trace := true;
-    let retval = (Meta.reify_bytecode code code_size) () in
+    let retval = Meta.reify_bytecode code code_size () in
     may_trace := false;
     if can_free then begin
       Meta.static_release_bytecode code code_size;
@@ -209,7 +209,7 @@
 (* The table of toplevel directives.
    Filled by functions from module topdirs. *)
 
-let directive_table = (Hashtbl.create 13 : (string, directive_fun) Hashtbl.t)
+let directive_table : (string, directive_fun) Hashtbl.t = Hashtbl.create 13
 
 (* Execute a toplevel phrase *)
 
@@ -365,12 +365,11 @@
     end else
       len
   end
-
+;;
 (* Toplevel initialization. Performed here instead of at the
    beginning of loop() so that user code linked in with ocamlmktop
    can call directives from Topdirs. *)
 
-let _ =
   Sys.interactive := true;
   let crc_intfs = Symtable.init_toplevel() in
   Compile.init_path();
@@ -378,6 +377,7 @@
     (fun (name, crc) ->
       Consistbl.set Env.crc_units name crc Sys.executable_name)
     crc_intfs
+;;
 
 let load_ocamlinit ppf =
   match !Clflags.init_file with
