--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/toplevel/opttopdirs.ml	2012-01-20 06:23:34.000000000 -0800
+++ patches/ocaml-4.00.0/ALL/toplevel!opttopdirs.ml.INPUT	2020-02-11 21:16:25.644514729 -0800
@@ -27,27 +27,28 @@
 (* To quit *)
 
 let dir_quit () = exit 0
-
-let _ = Hashtbl.add directive_table "quit" (Directive_none dir_quit)
+;;
+Hashtbl.add directive_table "quit" (Directive_none dir_quit) ;;
 
 (* To add a directory to the load path *)
 
 let dir_directory s =
   let d = expand_directory Config.standard_library s in
   Config.load_path := d :: !Config.load_path
+;;
 
-let _ = Hashtbl.add directive_table "directory" (Directive_string dir_directory)
-let _ = Hashtbl.add directive_table "show_dirs"
+Hashtbl.add directive_table "directory" (Directive_string dir_directory) ;;
+Hashtbl.add directive_table "show_dirs"
   (Directive_none
      (fun () ->
         List.iter print_endline !Config.load_path
      ))
-
+;;
 (* To change the current directory *)
 
 let dir_cd s = Sys.chdir s
-
-let _ = Hashtbl.add directive_table "cd" (Directive_string dir_cd)
+;;
+Hashtbl.add directive_table "cd" (Directive_string dir_cd) ;;
 
 (* Load in-core a .cmxs file *)
 
@@ -86,14 +87,14 @@
 
 
 let dir_load ppf name = ignore (load_file ppf name)
-
-let _ = Hashtbl.add directive_table "load" (Directive_string (dir_load std_out))
+;;
+Hashtbl.add directive_table "load" (Directive_string (dir_load std_out)) ;;
 
 (* Load commands from a file *)
 
 let dir_use ppf name = ignore(Opttoploop.use_file ppf name)
-
-let _ = Hashtbl.add directive_table "use" (Directive_string (dir_use std_out))
+;;
+Hashtbl.add directive_table "use" (Directive_string (dir_use std_out)) ;;
 
 (* Install, remove a printer *)
 
@@ -156,17 +157,17 @@
       fprintf ppf "No printer named %a.@." Printtyp.longident lid
     end
   with Exit -> ()
-
-let _ = Hashtbl.add directive_table "install_printer"
-             (Directive_ident (dir_install_printer std_out))
-let _ = Hashtbl.add directive_table "remove_printer"
-             (Directive_ident (dir_remove_printer std_out))
+;;
+Hashtbl.add directive_table "install_printer"
+             (Directive_ident (dir_install_printer std_out)) ;;
+Hashtbl.add directive_table "remove_printer"
+             (Directive_ident (dir_remove_printer std_out)) ;;
 
 let parse_warnings ppf iserr s =
   try Warnings.parse_options iserr s
   with Arg.Bad err -> fprintf ppf "%s.@." err
+;;
 
-let _ =
 (* Control the printing of values *)
 
   Hashtbl.add directive_table "print_depth"
@@ -187,3 +188,4 @@
 
   Hashtbl.add directive_table "warn_error"
              (Directive_string (parse_warnings std_out true))
+;;
