--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/typing/includemod.ml	2012-05-31 00:41:37.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/typing!includemod.ml.INPUT	2020-02-11 21:24:59.125229443 -0800
@@ -316,8 +316,9 @@
                     (Mtype.strengthen env mty1 path1) mty2)
   with Error reasons ->
     raise Not_found
+;;
 
-let _ = Env.check_modtype_inclusion := check_modtype_inclusion
+Env.check_modtype_inclusion := check_modtype_inclusion ;;
 
 (* Check that an implementation of a compilation unit meets its
    interface. *)
