--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/typing/env.ml	2012-07-13 01:49:06.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/typing!env.ml.INPUT	2020-02-11 21:24:05.673220134 -0800
@@ -258,8 +258,8 @@
     ps_filename: string;
     ps_flags: pers_flags list }
 
-let persistent_structures =
-  (Hashtbl.create 17 : (string, pers_struct option) Hashtbl.t)
+let persistent_structures : (string, pers_struct option) Hashtbl.t =
+  Hashtbl.create 17
 
 (* Consistency between persistent structures *)
 
@@ -1040,14 +1040,14 @@
     let comps = components_of_module f.fcomp_env f.fcomp_subst p mty in
     Hashtbl.add f.fcomp_cache p2 comps;
     comps
-
+;;
 (* Define forward functions *)
 
-let _ =
+
   components_of_module' := components_of_module;
   components_of_functor_appl' := components_of_functor_appl;
   components_of_module_maker' := components_of_module_maker
-
+;;
 (* Insertion of bindings by identifier *)
 
 let add_value ?check id desc env =
