--- /home/chet/Hack/Ocaml/GENERIC/4.00.0/.opam-switch/sources/ocaml-base-compiler.4.00.0/typing/predef.ml	2012-05-31 00:41:37.000000000 -0700
+++ patches/ocaml-4.00.0/ALL/typing!predef.ml.INPUT	2020-02-11 21:26:27.273223017 -0800
@@ -239,10 +239,10 @@
        ident_failure; ident_not_found; ident_sys_error; ident_end_of_file;
        ident_division_by_zero; ident_sys_blocked_io;
        ident_assert_failure; ident_undefined_recursive_module ]
-
+;;
 (* Start non-predef identifiers at 1000.  This way, more predefs can
    be defined in this file (above!) without breaking .cmi
    compatibility. *)
 
-let _ = Ident.set_current_time 999
+Ident.set_current_time 999 ;;
 let builtin_idents = List.rev !builtin_idents
