--- ../ocaml_src/main/pcaml.ml	2020-02-03 20:14:39.717859545 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_src!main!pcaml.ml.INPUT	2020-02-08 11:58:20.468931350 -0800
@@ -93,6 +93,7 @@
 let warning = ref warning_default_function;;
 let quotation_loc = ref None;;
 
+let _ =
 List.iter (fun (n, f) -> Quotation.add n f)
   ["id", Quotation.ExStr (fun _ s -> "$0:" ^ s ^ "$");
    "string", Quotation.ExStr (fun _ s -> "\"" ^ String.escaped s ^ "\"")];;
@@ -209,6 +210,7 @@
 
 let expr_eoi = Grammar.Entry.create gram "expr_eoi";;
 let patt_eoi = Grammar.Entry.create gram "patt_eoi";;
+let _ =
 Grammar.safe_extend
   [Grammar.extension (expr_eoi : 'expr_eoi Grammar.Entry.e) None
      [None, None,
@@ -465,6 +467,7 @@
 let vala_map f x = f x;;
 let vala_mapa f g x = f x;;
 
+let _ =
 add_option "-mode"
   (Arg.String
      (function
@@ -473,6 +476,7 @@
       | _ -> failwith "bad mode; use option -help for details"))
   "<mode> Set strict (S) or transitional (T) mode (bootstrapping only).";;
 
+let _ =
 add_option "-pmode"
   (Arg.Unit
      (fun () ->
@@ -481,5 +485,6 @@
         exit 0))
   "Print the current mode and exit.";;
 
+let _ =
 add_option "-dquot" (Arg.String (fun s -> Quotation.default := s))
   "<name> Set default quotation.";;
