--- ../ocaml_stuff/4.09.1/parsing/location.mli	2020-01-26 20:26:21.282098646 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!4.09.1!parsing!location.mli.INPUT	2020-02-08 15:02:48.516805060 -0800
@@ -25,7 +25,7 @@
 type t = Warnings.loc = {
   loc_start: Lexing.position;
   loc_end: Lexing.position;
-  loc_ghost: bool;
+  loc_ghost: bool
 }
 
 (** Note on the use of Lexing.position in this module.
@@ -62,7 +62,7 @@
 
 type 'a loc = {
   txt : 'a;
-  loc : t;
+  loc : t
 }
 
 val mknoloc : 'a -> 'a loc
@@ -115,7 +115,7 @@
 val msg: ?loc:t -> ('a, Format.formatter, unit, msg) format4 -> 'a
 
 type report_kind =
-  | Report_error
+    Report_error
   | Report_warning of string
   | Report_warning_as_error of string
   | Report_alert of string
@@ -124,7 +124,7 @@
 type report = {
   kind : report_kind;
   main : msg;
-  sub : msg list;
+  sub : msg list
 }
 
 type report_printer = {
@@ -145,7 +145,7 @@
   pp_submsg_loc : report_printer -> report ->
     Format.formatter -> t -> unit;
   pp_submsg_txt : report_printer -> report ->
-    Format.formatter -> (Format.formatter -> unit) -> unit;
+    Format.formatter -> (Format.formatter -> unit) -> unit
 }
 (** A printer for [report]s, defined using open-recursion.
     The goal is to make it easy to define new printers by re-using code from
