--- ../ocaml_stuff/3.09.3/utils/warnings.mli	2020-01-21 21:41:30.417906018 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!3.09.3!utils!warnings.mli.INPUT	2020-02-10 12:15:31.855791555 -0800
@@ -15,7 +15,7 @@
 open Format
 
 type t =                             (* A is all *)
-  | Comment_start                    (* C *)
+    Comment_start                    (* C *)
   | Comment_not_end
   | Deprecated                       (* D *)
   | Fragile_pat of string            (* E *)
