--- ../ocaml_src/meta/pa_extend.ml	2020-02-07 12:19:26.380043919 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_src!meta!pa_extend.ml.INPUT	2020-02-08 12:10:32.149650486 -0800
@@ -8,6 +8,7 @@
 
 let split_ext = ref false;;
 
+let _ =
 Pcaml.add_option "-split_ext" (Arg.Set split_ext)
   "Split EXTEND by using functions.";;
 
@@ -1867,6 +1868,7 @@
          | _ -> raise Stream.Failure)
 ;;
 
+let _ =
 Grammar.safe_extend
   (let _ = (expr : 'expr Grammar.Entry.e)
    and _ = (symbol : 'symbol Grammar.Entry.e)
@@ -2546,5 +2548,7 @@
            (fun (s : string) (loc : Ploc.t) ->
               (ATstring (loc, s) : 'string)))]]]);;
 
+let _ =
 Pcaml.add_option "-quotify" (Arg.Set quotify) "Generate code for quotations";;
+let _ =
 Pcaml.add_option "-meta_action" (Arg.Set meta_action) "Undocumented";;
