--- ../ocaml_stuff/3.03/utils/warnings.mli	2020-01-21 21:41:30.413906065 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!3.03!utils!warnings.mli.INPUT	2020-02-08 14:28:00.079101079 -0800
@@ -15,10 +15,8 @@
 open Format
 
 type t =                             (* A is all *)
-  | Comment of string                (* C *)
-  | Deprecated                       (* D *)
+    Comment of string                (* C *)
   | Partial_application              (* F *)
-  | Labels_omitted		     (* L *)
   | Method_override of string list   (* M *)
   | Partial_match of string          (* P *)
   | Statement_type                   (* S *)
@@ -27,7 +25,7 @@
   | Other of string                  (* X *)
 ;;
 
-val parse_options : bool -> string -> unit;;
+val parse_options : iserror:bool -> string -> unit;;
 
 val is_active : t -> bool;;
 val is_error : t -> bool;;
