--- ../ocaml_stuff/4.03.1/parsing/asttypes.mli	2020-01-21 21:41:30.425905928 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!4.03.1!parsing!asttypes.mli.INPUT	2020-02-10 12:06:46.217205993 -0800
@@ -48,11 +48,11 @@
 
 type 'a loc = 'a Location.loc = {
   txt : 'a;
-  loc : Location.t;
+  loc : Location.t
 }
 
 
 type variance =
-  | Covariant
+    Covariant
   | Contravariant
   | Invariant
