--- ../ocaml_stuff/2.99/utils/warnings.mli	2020-01-21 21:41:30.413906065 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!2.99!utils!warnings.mli.INPUT	2020-02-08 14:25:19.462090155 -0800
@@ -13,7 +13,7 @@
 (* Id *)
 
 type t =
-  | Partial_match of string          (* P *)
+    Partial_match of string          (* P *)
   | Unused_match                     (* U *)
   | Method_override of string list   (* M *)
   | Hide_instance_variable of string (* V *)
