--- ../ocaml_stuff/3.08.3/utils/warnings.mli	2020-01-21 21:41:30.417906018 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!3.08.3!utils!warnings.mli.INPUT	2020-02-08 14:34:46.859258347 -0800
@@ -15,7 +15,7 @@
 open Format
 
 type t =                             (* A is all *)
-  | Comment of string                (* C *)
+    Comment of string                (* C *)
   | Deprecated                       (* D *)
   | Fragile_pat of string            (* E *)
   | Partial_application              (* F *)
