--- ../ocaml_stuff/4.01.1/parsing/parsetree.mli	2020-01-21 21:41:30.425905928 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!4.01.1!parsing!parsetree.mli.INPUT	2020-02-10 11:38:08.709059513 -0800
@@ -162,12 +162,12 @@
 and class_signature = {
     pcsig_self: core_type;
     pcsig_fields: class_type_field list;
-    pcsig_loc: Location.t;
+    pcsig_loc: Location.t
   }
 
 and class_type_field = {
     pctf_desc: class_type_field_desc;
-    pctf_loc: Location.t;
+    pctf_loc: Location.t
   }
 
 and class_type_field_desc =
@@ -197,12 +197,12 @@
 
 and class_structure = {
     pcstr_pat: pattern;
-    pcstr_fields: class_field list;
+    psctr_fields: class_field list
   }
 
 and class_field = {
     pcf_desc: class_field_desc;
-    pcf_loc: Location.t;
+    pcf_loc: Location.t
   }
 
 and class_field_desc =
