--- ../ocaml_stuff/3.05/utils/warnings.mli	2020-01-21 21:41:30.413906065 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!3.05!utils!warnings.mli.INPUT	2020-02-08 14:28:58.957966321 -0800
@@ -15,7 +15,7 @@
 open Format
 
 type t =                             (* A is all *)
-  | Comment of string                (* C *)
+    Comment of string                (* C *)
   | Deprecated                       (* D *)
   | Partial_application              (* F *)
   | Labels_omitted                   (* L *)
