--- ../ocaml_stuff/4.10.0/utils/warnings.mli	2020-02-03 19:36:27.896107843 -0800
+++ patches/camlp5.7.11-ORIGINAL/ALL/ocaml_stuff!4.10.0!utils!warnings.mli.INPUT	2020-02-08 13:59:52.410197862 -0800
@@ -23,11 +23,11 @@
 type loc = {
   loc_start: Lexing.position;
   loc_end: Lexing.position;
-  loc_ghost: bool;
+  loc_ghost: bool
 }
 
 type t =
-  | Comment_start                           (*  1 *)
+    Comment_start                           (*  1 *)
   | Comment_not_end                         (*  2 *)
 (*| Deprecated --> alert "deprecated" *)    (*  3 *)
   | Fragile_match of string                 (*  4 *)
@@ -119,7 +119,7 @@
   { id : string
   ; message : string
   ; is_error : bool
-  ; sub_locs : (loc * string) list;
+  ; sub_locs : (loc * string) list
   }
 
 val report : t -> [ `Active of reporting_information | `Inactive ]
